// @ids-core/core/src/variables/js/deprecated-variables.umd.js
(function (root, factory) {
    if (typeof module === 'object' && module.exports) {
        module.exports = factory();
    } else if (typeof exports === 'object') {
        exports['_styleDictionary'] = factory();
    } else if (typeof define === 'function' && define.amd) {
        define([], factory);
    } else {
        root['_styleDictionary'] = factory();
    }
})(this, function () {
    return {
        layout: {
            grid: {
                'baseline-low': {
                    value: '0.25rem',
                    comment: '4px',
                    filePath: './src/tokens/baseline.json',
                    isSource: true,
                    original: {
                        value: '0.25rem',
                        comment: '4px',
                    },
                    name: 'LayoutGridBaselineLow',
                    attributes: {
                        category: 'layout',
                        type: 'grid',
                        item: 'baseline-low',
                    },
                    path: ['layout', 'grid', 'baseline-low'],
                },
                'baseline-high': {
                    value: '0.5rem',
                    comment: '8px',
                    filePath: './src/tokens/baseline.json',
                    isSource: true,
                    original: {
                        value: '0.5rem',
                        comment: '8px',
                    },
                    name: 'LayoutGridBaselineHigh',
                    attributes: {
                        category: 'layout',
                        type: 'grid',
                        item: 'baseline-high',
                    },
                    path: ['layout', 'grid', 'baseline-high'],
                },
                'baseline-low-px': {
                    value: '4px',
                    comment: '0.25rem',
                    filePath: './src/tokens/baseline.json',
                    isSource: true,
                    original: {
                        value: '4px',
                        comment: '0.25rem',
                    },
                    name: 'LayoutGridBaselineLowPx',
                    attributes: {
                        category: 'layout',
                        type: 'grid',
                        item: 'baseline-low-px',
                    },
                    path: ['layout', 'grid', 'baseline-low-px'],
                },
                'baseline-high-px': {
                    value: '8px',
                    comment: '0.5rem',
                    filePath: './src/tokens/baseline.json',
                    isSource: true,
                    original: {
                        value: '8px',
                        comment: '0.5rem',
                    },
                    name: 'LayoutGridBaselineHighPx',
                    attributes: {
                        category: 'layout',
                        type: 'grid',
                        item: 'baseline-high-px',
                    },
                    path: ['layout', 'grid', 'baseline-high-px'],
                },
            },
        },
    };
});
