// @ids-core/typography/src/variables/js/deprecated-variables.esm.js
export const AssetFontIfsansVariableName = 'If Sans';
export const AssetFontIfsansVariableEot = './fonts/IfSans-Variable.eot';
export const AssetFontIfsansVariableOtf = './fonts/IfSans-Variable.otf';
export const AssetFontIfsansVariableTtf = './fonts/IfSans-Variable.ttf';
export const AssetFontIfsansVariableWoff = './fonts/IfSans-Variable.woff';
export const AssetFontIfsansVariableWoff2 = './fonts/IfSans-Variable.woff2';
export const AssetFontIfsansVariableItalicName = 'If Sans Italic';
export const AssetFontIfsansVariableItalicEot = './fonts/IfSans-VariableItalic.eot';
export const AssetFontIfsansVariableItalicOtf = './fonts/IfSans-VariableItalic.otf';
export const AssetFontIfsansVariableItalicTtf = './fonts/IfSans-VariableItalic.ttf';
export const AssetFontIfsansVariableItalicWoff = './fonts/IfSans-VariableItalic.woff';
export const AssetFontIfsansVariableItalicWoff2 = './fonts/IfSans-VariableItalic.woff2';
export const AssetFontIfsansRegularName = 'If Sans';
export const AssetFontIfsansRegularEot = './fonts/IfSans-Regular.eot';
export const AssetFontIfsansRegularOtf = './fonts/IfSans-Regular.otf';
export const AssetFontIfsansRegularTtf = './fonts/IfSans-Regular.ttf';
export const AssetFontIfsansRegularWoff = './fonts/IfSans-Regular.woff';
export const AssetFontIfsansRegularWoff2 = './fonts/IfSans-Regular.woff2';
export const AssetFontIfsansThinName = 'If Sans Thin';
export const AssetFontIfsansThinEot = './fonts/IfSans-Thin.eot';
export const AssetFontIfsansThinOtf = './fonts/IfSans-Thin.otf';
export const AssetFontIfsansThinTtf = './fonts/IfSans-Thin.ttf';
export const AssetFontIfsansThinWoff = './fonts/IfSans-Thin.woff';
export const AssetFontIfsansThinWoff2 = './fonts/IfSans-Thin.woff2';
export const AssetFontIfsansLightName = 'If Sans Light';
export const AssetFontIfsansLightEot = './fonts/IfSans-Light.eot';
export const AssetFontIfsansLightOtf = './fonts/IfSans-Light.otf';
export const AssetFontIfsansLightTtf = './fonts/IfSans-Light.ttf';
export const AssetFontIfsansLightWoff = './fonts/IfSans-Light.woff';
export const AssetFontIfsansLightWoff2 = './fonts/IfSans-Light.woff2';
export const AssetFontIfsansMediumName = 'If Sans Medium';
export const AssetFontIfsansMediumEot = './fonts/IfSans-Medium.eot';
export const AssetFontIfsansMediumOtf = './fonts/IfSans-Medium.otf';
export const AssetFontIfsansMediumTtf = './fonts/IfSans-Medium.ttf';
export const AssetFontIfsansMediumWoff = './fonts/IfSans-Medium.woff';
export const AssetFontIfsansMediumWoff2 = './fonts/IfSans-Medium.woff2';
export const AssetFontIfsansBoldName = 'If Sans Bold';
export const AssetFontIfsansBoldEot = './fonts/IfSans-Bold.eot';
export const AssetFontIfsansBoldOtf = './fonts/IfSans-Bold.otf';
export const AssetFontIfsansBoldTtf = './fonts/IfSans-Bold.ttf';
export const AssetFontIfsansBoldWoff = './fonts/IfSans-Bold.woff';
export const AssetFontIfsansBoldWoff2 = './fonts/IfSans-Bold.woff2';
export const AssetFontIfsansRegularItalicName = 'If Sans Italic';
export const AssetFontIfsansRegularItalicEot = './fonts/IfSans-Italic.eot';
export const AssetFontIfsansRegularItalicOtf = './fonts/IfSans-Italic.otf';
export const AssetFontIfsansRegularItalicTtf = './fonts/IfSans-Italic.ttf';
export const AssetFontIfsansRegularItalicWoff = './fonts/IfSans-Italic.woff';
export const AssetFontIfsansRegularItalicWoff2 = './fonts/IfSans-Italic.woff2';
export const AssetFontIfsansThinItalicName = 'If Sans Thin Italic';
export const AssetFontIfsansThinItalicEot = './fonts/IfSans-ThinItalic.eot';
export const AssetFontIfsansThinItalicOtf = './fonts/IfSans-ThinItalic.otf';
export const AssetFontIfsansThinItalicTtf = './fonts/IfSans-ThinItalic.ttf';
export const AssetFontIfsansThinItalicWoff = './fonts/IfSans-ThinItalic.woff';
export const AssetFontIfsansThinItalicWoff2 = './fonts/IfSans-ThinItalic.woff2';
export const AssetFontIfsansLightItalicName = 'If Sans Light Italic';
export const AssetFontIfsansLightItalicEot = './fonts/IfSans-LightItalic.eot';
export const AssetFontIfsansLightItalicOtf = './fonts/IfSans-LightItalic.otf';
export const AssetFontIfsansLightItalicTtf = './fonts/IfSans-LightItalic.ttf';
export const AssetFontIfsansLightItalicWoff = './fonts/IfSans-LightItalic.woff';
export const AssetFontIfsansLightItalicWoff2 = './fonts/IfSans-LightItalic.woff2';
export const AssetFontIfsansMediumItalicName = 'If Sans Medium Italic';
export const AssetFontIfsansMediumItalicEot = './fonts/IfSans-MediumItalic.eot';
export const AssetFontIfsansMediumItalicOtf = './fonts/IfSans-MediumItalic.otf';
export const AssetFontIfsansMediumItalicTtf = './fonts/IfSans-MediumItalic.ttf';
export const AssetFontIfsansMediumItalicWoff = './fonts/IfSans-MediumItalic.woff';
export const AssetFontIfsansMediumItalicWoff2 = './fonts/IfSans-MediumItalic.woff2';
export const AssetFontIfsansBoldItalicName = 'If Sans Bold Italic';
export const AssetFontIfsansBoldItalicEot = './fonts/IfSans-BoldItalic.eot';
export const AssetFontIfsansBoldItalicOtf = './fonts/IfSans-BoldItalic.otf';
export const AssetFontIfsansBoldItalicTtf = './fonts/IfSans-BoldItalic.ttf';
export const AssetFontIfsansBoldItalicWoff = './fonts/IfSans-BoldItalic.woff';
export const AssetFontIfsansBoldItalicWoff2 = './fonts/IfSans-BoldItalic.woff2';
export const AssetFontIfsansVariableThinName = 'If Sans Thin';
export const AssetFontIfsansVariableThinEot = './fonts/IfSans-Variable.eot';
export const AssetFontIfsansVariableThinOtf = './fonts/IfSans-Variable.otf';
export const AssetFontIfsansVariableThinTtf = './fonts/IfSans-Variable.ttf';
export const AssetFontIfsansVariableThinWoff = './fonts/IfSans-Variable.woff';
export const AssetFontIfsansVariableThinWoff2 = './fonts/IfSans-Variable.woff2';
export const AssetFontIfsansVariableLightName = 'If Sans Light';
export const AssetFontIfsansVariableLightEot = './fonts/IfSans-Variable.eot';
export const AssetFontIfsansVariableLightOtf = './fonts/IfSans-Variable.otf';
export const AssetFontIfsansVariableLightTtf = './fonts/IfSans-Variable.ttf';
export const AssetFontIfsansVariableLightWoff = './fonts/IfSans-Variable.woff';
export const AssetFontIfsansVariableLightWoff2 = './fonts/IfSans-Variable.woff2';
export const AssetFontIfsansVariableMediumName = 'If Sans Medium';
export const AssetFontIfsansVariableMediumEot = './fonts/IfSans-Variable.eot';
export const AssetFontIfsansVariableMediumOtf = './fonts/IfSans-Variable.otf';
export const AssetFontIfsansVariableMediumTtf = './fonts/IfSans-Variable.ttf';
export const AssetFontIfsansVariableMediumWoff = './fonts/IfSans-Variable.woff';
export const AssetFontIfsansVariableMediumWoff2 = './fonts/IfSans-Variable.woff2';
export const AssetFontIfsansVariableBoldName = 'If Sans Bold';
export const AssetFontIfsansVariableBoldEot = './fonts/IfSans-Variable.eot';
export const AssetFontIfsansVariableBoldOtf = './fonts/IfSans-Variable.otf';
export const AssetFontIfsansVariableBoldTtf = './fonts/IfSans-Variable.ttf';
export const AssetFontIfsansVariableBoldWoff = './fonts/IfSans-Variable.woff';
export const AssetFontIfsansVariableBoldWoff2 = './fonts/IfSans-Variable.woff2';
export const AssetFontIfsansVariableThinItalicName = 'If Sans Thin Italic';
export const AssetFontIfsansVariableThinItalicEot = './fonts/IfSans-VariableItalic.eot';
export const AssetFontIfsansVariableThinItalicOtf = './fonts/IfSans-VariableItalic.otf';
export const AssetFontIfsansVariableThinItalicTtf = './fonts/IfSans-VariableItalic.ttf';
export const AssetFontIfsansVariableThinItalicWoff = './fonts/IfSans-VariableItalic.woff';
export const AssetFontIfsansVariableThinItalicWoff2 = './fonts/IfSans-VariableItalic.woff2';
export const AssetFontIfsansVariableLightItalicName = 'If Sans Light Italic';
export const AssetFontIfsansVariableLightItalicEot = './fonts/IfSans-VariableItalic.eot';
export const AssetFontIfsansVariableLightItalicOtf = './fonts/IfSans-VariableItalic.otf';
export const AssetFontIfsansVariableLightItalicTtf = './fonts/IfSans-VariableItalic.ttf';
export const AssetFontIfsansVariableLightItalicWoff = './fonts/IfSans-VariableItalic.woff';
export const AssetFontIfsansVariableLightItalicWoff2 = './fonts/IfSans-VariableItalic.woff2';
export const AssetFontIfsansVariableMediumItalicName = 'If Sans Medium Italic';
export const AssetFontIfsansVariableMediumItalicEot = './fonts/IfSans-VariableItalic.eot';
export const AssetFontIfsansVariableMediumItalicOtf = './fonts/IfSans-VariableItalic.otf';
export const AssetFontIfsansVariableMediumItalicTtf = './fonts/IfSans-VariableItalic.ttf';
export const AssetFontIfsansVariableMediumItalicWoff = './fonts/IfSans-VariableItalic.woff';
export const AssetFontIfsansVariableMediumItalicWoff2 = './fonts/IfSans-VariableItalic.woff2';
export const AssetFontIfsansVariableBoldItalicName = 'If Sans Bold Italic';
export const AssetFontIfsansVariableBoldItalicEot = './fonts/IfSans-VariableItalic.eot';
export const AssetFontIfsansVariableBoldItalicOtf = './fonts/IfSans-VariableItalic.otf';
export const AssetFontIfsansVariableBoldItalicTtf = './fonts/IfSans-VariableItalic.ttf';
export const AssetFontIfsansVariableBoldItalicWoff = './fonts/IfSans-VariableItalic.woff';
export const AssetFontIfsansVariableBoldItalicWoff2 = './fonts/IfSans-VariableItalic.woff2';
export const SizeFont14 = '0.875rem'; // 14 pixels
export const SizeFont15 = '0.9375rem'; // 15 pixels
export const SizeFont16 = '1rem'; // 16 pixels
export const SizeFont18 = '1.125rem'; // 18 pixels
export const SizeFont20 = '1.25rem'; // 20 pixels
export const SizeFont22 = '1.375rem'; // 22 pixels
export const SizeFont24 = '1.5rem'; // 24 pixels
export const SizeFont26 = '1.625rem'; // 26 pixels
export const SizeFont28 = '1.75rem'; // 28 pixels
export const SizeFont30 = '1.875rem'; // 30 pixels
export const SizeFont32 = '2rem'; // 32 pixels
export const SizeFont33 = '2.0625rem'; // 33 pixels
export const SizeFont36 = '2.25rem'; // 36 pixels
export const SizeFont38 = '2.375rem'; // 38 pixels
export const SizeFont42 = '2.625rem'; // 42 pixels
export const SizeFont48 = '3rem'; // 48 pixels
export const SizeFont54 = '3.375rem'; // 54 pixels
export const SizeFont56 = '3.5rem'; // 56 pixels
export const SizeFont66 = '4.125rem'; // 66 pixels
export const SizeFont14px = 14; // 0.875rem
export const SizeFont15px = 15; // 0.9375rem
export const SizeFont16px = 16; // 1rem
export const SizeFont18px = 18; // 1.125rem
export const SizeFont20px = 20; // 1.25rem
export const SizeFont22px = 22; // 1.375rem
export const SizeFont24px = 24; // 1.5rem
export const SizeFont26px = 26; // 1.625rem
export const SizeFont28px = 28; // 1.75rem
export const SizeFont30px = 30; // 1.875rem
export const SizeFont32px = 32; // 2rem
export const SizeFont33px = 33; // 2.0625rem
export const SizeFont36px = 36; // 2.25rem
export const SizeFont38px = 38; // 2.375rem
export const SizeFont42px = 42; // 2.625rem
export const SizeFont48px = 48; // 3rem
export const SizeFont54px = 54; // 3.375rem
export const SizeFont56px = 56; // 3.5rem
export const SizeFont66px = 66; // 4.125rem
export const SizeLineHeight24 = '1.5rem'; // 24px
export const SizeLineHeight28 = '1.75rem'; // 28px
export const SizeLineHeight32 = '2rem'; // 32px
export const SizeLineHeight40 = '2.5rem'; // 40px
export const SizeLineHeight42 = '2.625rem'; // 42px
export const SizeLineHeight44 = '2.75rem'; // 44px
export const SizeLineHeight48 = '3rem'; // 48px
export const SizeLineHeight52 = '3.25rem'; // 52px
export const SizeLineHeight56 = '3.5rem'; // 56px
export const SizeLineHeight64 = '4rem'; // 64px
export const SizeLineHeight72 = '4.5rem'; // 72px
export const SizeLineHeight24px = 24; // 1.5rem
export const SizeLineHeight28px = 28; // 1.75rem
export const SizeLineHeight32px = 32; // 2rem
export const SizeLineHeight40px = 40; // 2.5rem
export const SizeLineHeight42px = 42; // 2.625rem
export const SizeLineHeight44px = 44; // 2.75rem
export const SizeLineHeight48px = 48; // 3rem
export const SizeLineHeight52px = 52; // 3.25rem
export const SizeLineHeight56px = 56; // 3.5rem
export const SizeLineHeight64px = 64; // 4rem
export const SizeLineHeight72px = 72; // 4.5rem
export const FontWeight36 = 36;
export const FontWeight40 = 40;
export const FontWeight45 = 45;
export const FontWeight48 = 48;
export const FontWeight50 = 50;
export const FontWeight54 = 54;
export const FontWeight64 = 64;
export const FontWeight70 = 70;
export const FontWeight78 = 78;
export const FontWeight82 = 82;
export const FontWeight85 = 85;
export const FontWeight104 = 104;
export const FontWeight126 = 126;
export const FontFamilySansThin = 'If Sans Thin';
export const FontFamilySansLight = 'If Sans Light';
export const FontFamilySans = 'If Sans';
export const FontFamilySansMedium = 'If Sans Medium';
export const FontFamilySansBold = 'If Sans Bold';
export const FontFamilySansThinItalic = 'If Sans Thin Italic';
export const FontFamilySansLightItalic = 'If Sans Light Italic';
export const FontFamilySansItalic = 'If Sans Italic';
export const FontFamilySansMediumItalic = 'If Sans Medium Italic';
export const FontFamilySansBoldItalic = 'If Sans Bold Italic';
