// @ids-core/typography/src/variables/js/variables.umd.js
const generated_variables = require('./generated-variables.umd.js');
/*
 * Deprecated variables
 * @todo remove in version 16
 */
const deprecated_variables = {
    asset: {
        font: {
            'ifsans-variable': {
                name: {
                    value: 'If Sans',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: 'If Sans',
                    },
                    name: 'AssetFontIfsansVariableName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-variable', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-Variable.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Variable.eot',
                    },
                    name: 'AssetFontIfsansVariableEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-variable', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-Variable.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Variable.otf',
                    },
                    name: 'AssetFontIfsansVariableOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-variable', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-Variable.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Variable.ttf',
                    },
                    name: 'AssetFontIfsansVariableTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-variable', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-Variable.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Variable.woff',
                    },
                    name: 'AssetFontIfsansVariableWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-variable', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-Variable.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Variable.woff2',
                    },
                    name: 'AssetFontIfsansVariableWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-variable', 'woff2'],
                },
            },
            'ifsans-variable-italic': {
                name: {
                    value: 'If Sans Italic',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: 'If Sans Italic',
                    },
                    name: 'AssetFontIfsansVariableItalicName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-italic',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-variable-italic', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-VariableItalic.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-VariableItalic.eot',
                    },
                    name: 'AssetFontIfsansVariableItalicEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-italic',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-variable-italic', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-VariableItalic.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-VariableItalic.otf',
                    },
                    name: 'AssetFontIfsansVariableItalicOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-italic',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-italic', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-VariableItalic.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-VariableItalic.ttf',
                    },
                    name: 'AssetFontIfsansVariableItalicTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-italic',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-italic', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-VariableItalic.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-VariableItalic.woff',
                    },
                    name: 'AssetFontIfsansVariableItalicWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-italic',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-variable-italic', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-VariableItalic.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-VariableItalic.woff2',
                    },
                    name: 'AssetFontIfsansVariableItalicWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-italic',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-variable-italic', 'woff2'],
                },
            },
            'ifsans-regular': {
                name: {
                    value: 'If Sans',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: 'If Sans',
                    },
                    name: 'AssetFontIfsansRegularName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-regular',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-regular', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-Regular.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Regular.eot',
                    },
                    name: 'AssetFontIfsansRegularEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-regular',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-regular', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-Regular.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Regular.otf',
                    },
                    name: 'AssetFontIfsansRegularOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-regular',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-regular', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-Regular.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Regular.ttf',
                    },
                    name: 'AssetFontIfsansRegularTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-regular',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-regular', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-Regular.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Regular.woff',
                    },
                    name: 'AssetFontIfsansRegularWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-regular',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-regular', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-Regular.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Regular.woff2',
                    },
                    name: 'AssetFontIfsansRegularWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-regular',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-regular', 'woff2'],
                },
            },
            'ifsans-thin': {
                name: {
                    value: 'If Sans Thin',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: 'If Sans Thin',
                    },
                    name: 'AssetFontIfsansThinName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-thin',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-thin', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-Thin.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Thin.eot',
                    },
                    name: 'AssetFontIfsansThinEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-thin',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-thin', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-Thin.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Thin.otf',
                    },
                    name: 'AssetFontIfsansThinOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-thin',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-thin', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-Thin.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Thin.ttf',
                    },
                    name: 'AssetFontIfsansThinTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-thin',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-thin', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-Thin.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Thin.woff',
                    },
                    name: 'AssetFontIfsansThinWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-thin',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-thin', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-Thin.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Thin.woff2',
                    },
                    name: 'AssetFontIfsansThinWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-thin',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-thin', 'woff2'],
                },
            },
            'ifsans-light': {
                name: {
                    value: 'If Sans Light',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: 'If Sans Light',
                    },
                    name: 'AssetFontIfsansLightName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-light',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-light', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-Light.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Light.eot',
                    },
                    name: 'AssetFontIfsansLightEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-light',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-light', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-Light.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Light.otf',
                    },
                    name: 'AssetFontIfsansLightOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-light',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-light', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-Light.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Light.ttf',
                    },
                    name: 'AssetFontIfsansLightTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-light',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-light', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-Light.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Light.woff',
                    },
                    name: 'AssetFontIfsansLightWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-light',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-light', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-Light.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Light.woff2',
                    },
                    name: 'AssetFontIfsansLightWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-light',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-light', 'woff2'],
                },
            },
            'ifsans-medium': {
                name: {
                    value: 'If Sans Medium',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: 'If Sans Medium',
                    },
                    name: 'AssetFontIfsansMediumName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-medium',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-medium', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-Medium.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Medium.eot',
                    },
                    name: 'AssetFontIfsansMediumEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-medium',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-medium', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-Medium.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Medium.otf',
                    },
                    name: 'AssetFontIfsansMediumOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-medium',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-medium', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-Medium.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Medium.ttf',
                    },
                    name: 'AssetFontIfsansMediumTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-medium',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-medium', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-Medium.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Medium.woff',
                    },
                    name: 'AssetFontIfsansMediumWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-medium',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-medium', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-Medium.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Medium.woff2',
                    },
                    name: 'AssetFontIfsansMediumWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-medium',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-medium', 'woff2'],
                },
            },
            'ifsans-bold': {
                name: {
                    value: 'If Sans Bold',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: 'If Sans Bold',
                    },
                    name: 'AssetFontIfsansBoldName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-bold',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-bold', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-Bold.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Bold.eot',
                    },
                    name: 'AssetFontIfsansBoldEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-bold',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-bold', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-Bold.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Bold.otf',
                    },
                    name: 'AssetFontIfsansBoldOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-bold',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-bold', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-Bold.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Bold.ttf',
                    },
                    name: 'AssetFontIfsansBoldTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-bold',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-bold', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-Bold.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Bold.woff',
                    },
                    name: 'AssetFontIfsansBoldWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-bold',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-bold', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-Bold.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Bold.woff2',
                    },
                    name: 'AssetFontIfsansBoldWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-bold',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-bold', 'woff2'],
                },
            },
            'ifsans-regular-italic': {
                name: {
                    value: 'If Sans Italic',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: 'If Sans Italic',
                    },
                    name: 'AssetFontIfsansRegularItalicName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-regular-italic',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-regular-italic', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-Italic.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Italic.eot',
                    },
                    name: 'AssetFontIfsansRegularItalicEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-regular-italic',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-regular-italic', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-Italic.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Italic.otf',
                    },
                    name: 'AssetFontIfsansRegularItalicOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-regular-italic',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-regular-italic', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-Italic.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Italic.ttf',
                    },
                    name: 'AssetFontIfsansRegularItalicTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-regular-italic',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-regular-italic', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-Italic.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Italic.woff',
                    },
                    name: 'AssetFontIfsansRegularItalicWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-regular-italic',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-regular-italic', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-Italic.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-Italic.woff2',
                    },
                    name: 'AssetFontIfsansRegularItalicWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-regular-italic',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-regular-italic', 'woff2'],
                },
            },
            'ifsans-thin-italic': {
                name: {
                    value: 'If Sans Thin Italic',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: 'If Sans Thin Italic',
                    },
                    name: 'AssetFontIfsansThinItalicName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-thin-italic',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-thin-italic', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-ThinItalic.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-ThinItalic.eot',
                    },
                    name: 'AssetFontIfsansThinItalicEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-thin-italic',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-thin-italic', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-ThinItalic.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-ThinItalic.otf',
                    },
                    name: 'AssetFontIfsansThinItalicOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-thin-italic',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-thin-italic', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-ThinItalic.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-ThinItalic.ttf',
                    },
                    name: 'AssetFontIfsansThinItalicTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-thin-italic',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-thin-italic', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-ThinItalic.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-ThinItalic.woff',
                    },
                    name: 'AssetFontIfsansThinItalicWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-thin-italic',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-thin-italic', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-ThinItalic.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-ThinItalic.woff2',
                    },
                    name: 'AssetFontIfsansThinItalicWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-thin-italic',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-thin-italic', 'woff2'],
                },
            },
            'ifsans-light-italic': {
                name: {
                    value: 'If Sans Light Italic',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: 'If Sans Light Italic',
                    },
                    name: 'AssetFontIfsansLightItalicName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-light-italic',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-light-italic', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-LightItalic.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-LightItalic.eot',
                    },
                    name: 'AssetFontIfsansLightItalicEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-light-italic',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-light-italic', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-LightItalic.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-LightItalic.otf',
                    },
                    name: 'AssetFontIfsansLightItalicOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-light-italic',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-light-italic', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-LightItalic.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-LightItalic.ttf',
                    },
                    name: 'AssetFontIfsansLightItalicTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-light-italic',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-light-italic', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-LightItalic.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-LightItalic.woff',
                    },
                    name: 'AssetFontIfsansLightItalicWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-light-italic',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-light-italic', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-LightItalic.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-LightItalic.woff2',
                    },
                    name: 'AssetFontIfsansLightItalicWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-light-italic',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-light-italic', 'woff2'],
                },
            },
            'ifsans-medium-italic': {
                name: {
                    value: 'If Sans Medium Italic',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: 'If Sans Medium Italic',
                    },
                    name: 'AssetFontIfsansMediumItalicName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-medium-italic',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-medium-italic', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-MediumItalic.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-MediumItalic.eot',
                    },
                    name: 'AssetFontIfsansMediumItalicEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-medium-italic',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-medium-italic', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-MediumItalic.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-MediumItalic.otf',
                    },
                    name: 'AssetFontIfsansMediumItalicOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-medium-italic',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-medium-italic', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-MediumItalic.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-MediumItalic.ttf',
                    },
                    name: 'AssetFontIfsansMediumItalicTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-medium-italic',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-medium-italic', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-MediumItalic.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-MediumItalic.woff',
                    },
                    name: 'AssetFontIfsansMediumItalicWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-medium-italic',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-medium-italic', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-MediumItalic.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-MediumItalic.woff2',
                    },
                    name: 'AssetFontIfsansMediumItalicWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-medium-italic',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-medium-italic', 'woff2'],
                },
            },
            'ifsans-bold-italic': {
                name: {
                    value: 'If Sans Bold Italic',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: 'If Sans Bold Italic',
                    },
                    name: 'AssetFontIfsansBoldItalicName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-bold-italic',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-bold-italic', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-BoldItalic.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-BoldItalic.eot',
                    },
                    name: 'AssetFontIfsansBoldItalicEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-bold-italic',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-bold-italic', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-BoldItalic.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-BoldItalic.otf',
                    },
                    name: 'AssetFontIfsansBoldItalicOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-bold-italic',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-bold-italic', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-BoldItalic.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-BoldItalic.ttf',
                    },
                    name: 'AssetFontIfsansBoldItalicTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-bold-italic',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-bold-italic', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-BoldItalic.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-BoldItalic.woff',
                    },
                    name: 'AssetFontIfsansBoldItalicWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-bold-italic',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-bold-italic', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-BoldItalic.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: './fonts/IfSans-BoldItalic.woff2',
                    },
                    name: 'AssetFontIfsansBoldItalicWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-bold-italic',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-bold-italic', 'woff2'],
                },
            },
            'ifsans-variable-thin': {
                name: {
                    value: 'If Sans Thin',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-thin.name.value}',
                    },
                    name: 'AssetFontIfsansVariableThinName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-thin',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-variable-thin', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-Variable.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.eot.value}',
                    },
                    name: 'AssetFontIfsansVariableThinEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-thin',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-variable-thin', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-Variable.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.otf.value}',
                    },
                    name: 'AssetFontIfsansVariableThinOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-thin',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-thin', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-Variable.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.ttf.value}',
                    },
                    name: 'AssetFontIfsansVariableThinTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-thin',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-thin', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-Variable.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.woff.value}',
                    },
                    name: 'AssetFontIfsansVariableThinWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-thin',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-variable-thin', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-Variable.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.woff2.value}',
                    },
                    name: 'AssetFontIfsansVariableThinWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-thin',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-variable-thin', 'woff2'],
                },
            },
            'ifsans-variable-light': {
                name: {
                    value: 'If Sans Light',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-light.name.value}',
                    },
                    name: 'AssetFontIfsansVariableLightName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-light',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-variable-light', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-Variable.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.eot.value}',
                    },
                    name: 'AssetFontIfsansVariableLightEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-light',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-variable-light', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-Variable.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.otf.value}',
                    },
                    name: 'AssetFontIfsansVariableLightOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-light',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-light', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-Variable.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.ttf.value}',
                    },
                    name: 'AssetFontIfsansVariableLightTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-light',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-light', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-Variable.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.woff.value}',
                    },
                    name: 'AssetFontIfsansVariableLightWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-light',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-variable-light', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-Variable.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.woff2.value}',
                    },
                    name: 'AssetFontIfsansVariableLightWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-light',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-variable-light', 'woff2'],
                },
            },
            'ifsans-variable-medium': {
                name: {
                    value: 'If Sans Medium',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-medium.name.value}',
                    },
                    name: 'AssetFontIfsansVariableMediumName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-medium',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-variable-medium', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-Variable.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.eot.value}',
                    },
                    name: 'AssetFontIfsansVariableMediumEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-medium',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-variable-medium', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-Variable.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.otf.value}',
                    },
                    name: 'AssetFontIfsansVariableMediumOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-medium',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-medium', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-Variable.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.ttf.value}',
                    },
                    name: 'AssetFontIfsansVariableMediumTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-medium',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-medium', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-Variable.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.woff.value}',
                    },
                    name: 'AssetFontIfsansVariableMediumWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-medium',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-variable-medium', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-Variable.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.woff2.value}',
                    },
                    name: 'AssetFontIfsansVariableMediumWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-medium',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-variable-medium', 'woff2'],
                },
            },
            'ifsans-variable-bold': {
                name: {
                    value: 'If Sans Bold',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-bold.name.value}',
                    },
                    name: 'AssetFontIfsansVariableBoldName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-bold',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-variable-bold', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-Variable.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.eot.value}',
                    },
                    name: 'AssetFontIfsansVariableBoldEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-bold',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-variable-bold', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-Variable.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.otf.value}',
                    },
                    name: 'AssetFontIfsansVariableBoldOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-bold',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-bold', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-Variable.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.ttf.value}',
                    },
                    name: 'AssetFontIfsansVariableBoldTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-bold',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-bold', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-Variable.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.woff.value}',
                    },
                    name: 'AssetFontIfsansVariableBoldWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-bold',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-variable-bold', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-Variable.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable.woff2.value}',
                    },
                    name: 'AssetFontIfsansVariableBoldWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-bold',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-variable-bold', 'woff2'],
                },
            },
            'ifsans-variable-thin-italic': {
                name: {
                    value: 'If Sans Thin Italic',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-thin-italic.name.value}',
                    },
                    name: 'AssetFontIfsansVariableThinItalicName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-thin-italic',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-variable-thin-italic', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-VariableItalic.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.eot.value}',
                    },
                    name: 'AssetFontIfsansVariableThinItalicEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-thin-italic',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-variable-thin-italic', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-VariableItalic.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.otf.value}',
                    },
                    name: 'AssetFontIfsansVariableThinItalicOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-thin-italic',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-thin-italic', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-VariableItalic.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.ttf.value}',
                    },
                    name: 'AssetFontIfsansVariableThinItalicTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-thin-italic',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-thin-italic', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-VariableItalic.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.woff.value}',
                    },
                    name: 'AssetFontIfsansVariableThinItalicWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-thin-italic',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-variable-thin-italic', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-VariableItalic.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.woff2.value}',
                    },
                    name: 'AssetFontIfsansVariableThinItalicWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-thin-italic',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-variable-thin-italic', 'woff2'],
                },
            },
            'ifsans-variable-light-italic': {
                name: {
                    value: 'If Sans Light Italic',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-light-italic.name.value}',
                    },
                    name: 'AssetFontIfsansVariableLightItalicName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-light-italic',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-variable-light-italic', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-VariableItalic.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.eot.value}',
                    },
                    name: 'AssetFontIfsansVariableLightItalicEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-light-italic',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-variable-light-italic', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-VariableItalic.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.otf.value}',
                    },
                    name: 'AssetFontIfsansVariableLightItalicOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-light-italic',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-light-italic', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-VariableItalic.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.ttf.value}',
                    },
                    name: 'AssetFontIfsansVariableLightItalicTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-light-italic',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-light-italic', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-VariableItalic.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.woff.value}',
                    },
                    name: 'AssetFontIfsansVariableLightItalicWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-light-italic',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-variable-light-italic', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-VariableItalic.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.woff2.value}',
                    },
                    name: 'AssetFontIfsansVariableLightItalicWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-light-italic',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-variable-light-italic', 'woff2'],
                },
            },
            'ifsans-variable-medium-italic': {
                name: {
                    value: 'If Sans Medium Italic',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-medium-italic.name.value}',
                    },
                    name: 'AssetFontIfsansVariableMediumItalicName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-medium-italic',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-variable-medium-italic', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-VariableItalic.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.eot.value}',
                    },
                    name: 'AssetFontIfsansVariableMediumItalicEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-medium-italic',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-variable-medium-italic', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-VariableItalic.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.otf.value}',
                    },
                    name: 'AssetFontIfsansVariableMediumItalicOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-medium-italic',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-medium-italic', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-VariableItalic.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.ttf.value}',
                    },
                    name: 'AssetFontIfsansVariableMediumItalicTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-medium-italic',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-medium-italic', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-VariableItalic.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.woff.value}',
                    },
                    name: 'AssetFontIfsansVariableMediumItalicWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-medium-italic',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-variable-medium-italic', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-VariableItalic.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.woff2.value}',
                    },
                    name: 'AssetFontIfsansVariableMediumItalicWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-medium-italic',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-variable-medium-italic', 'woff2'],
                },
            },
            'ifsans-variable-bold-italic': {
                name: {
                    value: 'If Sans Bold Italic',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-bold-italic.name.value}',
                    },
                    name: 'AssetFontIfsansVariableBoldItalicName',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-bold-italic',
                        subitem: 'name',
                    },
                    path: ['asset', 'font', 'ifsans-variable-bold-italic', 'name'],
                },
                eot: {
                    value: './fonts/IfSans-VariableItalic.eot',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.eot.value}',
                    },
                    name: 'AssetFontIfsansVariableBoldItalicEot',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-bold-italic',
                        subitem: 'eot',
                    },
                    path: ['asset', 'font', 'ifsans-variable-bold-italic', 'eot'],
                },
                otf: {
                    value: './fonts/IfSans-VariableItalic.otf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.otf.value}',
                    },
                    name: 'AssetFontIfsansVariableBoldItalicOtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-bold-italic',
                        subitem: 'otf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-bold-italic', 'otf'],
                },
                ttf: {
                    value: './fonts/IfSans-VariableItalic.ttf',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.ttf.value}',
                    },
                    name: 'AssetFontIfsansVariableBoldItalicTtf',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-bold-italic',
                        subitem: 'ttf',
                    },
                    path: ['asset', 'font', 'ifsans-variable-bold-italic', 'ttf'],
                },
                woff: {
                    value: './fonts/IfSans-VariableItalic.woff',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.woff.value}',
                    },
                    name: 'AssetFontIfsansVariableBoldItalicWoff',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-bold-italic',
                        subitem: 'woff',
                    },
                    path: ['asset', 'font', 'ifsans-variable-bold-italic', 'woff'],
                },
                woff2: {
                    value: './fonts/IfSans-VariableItalic.woff2',
                    filePath: './src/tokens/asset.json',
                    isSource: true,
                    original: {
                        value: '{asset.font.ifsans-variable-italic.woff2.value}',
                    },
                    name: 'AssetFontIfsansVariableBoldItalicWoff2',
                    attributes: {
                        category: 'asset',
                        type: 'font',
                        item: 'ifsans-variable-bold-italic',
                        subitem: 'woff2',
                    },
                    path: ['asset', 'font', 'ifsans-variable-bold-italic', 'woff2'],
                },
            },
        },
    },
    size: {
        font: {
            14: {
                value: '0.875rem',
                target: 'mobile',
                comment: '14 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '0.875rem',
                    target: 'mobile',
                    comment: '14 pixels',
                },
                name: 'SizeFont14',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '14',
                },
                path: ['size', 'font', '14'],
            },
            15: {
                value: '0.9375rem',
                target: 'mobile',
                comment: '15 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '0.9375rem',
                    target: 'mobile',
                    comment: '15 pixels',
                },
                name: 'SizeFont15',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '15',
                },
                path: ['size', 'font', '15'],
            },
            16: {
                value: '1rem',
                target: 'mobile',
                comment: '16 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '1rem',
                    target: 'mobile',
                    comment: '16 pixels',
                },
                name: 'SizeFont16',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '16',
                },
                path: ['size', 'font', '16'],
            },
            18: {
                value: '1.125rem',
                target: 'mobile',
                comment: '18 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '1.125rem',
                    target: 'mobile',
                    comment: '18 pixels',
                },
                name: 'SizeFont18',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '18',
                },
                path: ['size', 'font', '18'],
            },
            20: {
                value: '1.25rem',
                target: 'mobile',
                comment: '20 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '1.25rem',
                    target: 'mobile',
                    comment: '20 pixels',
                },
                name: 'SizeFont20',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '20',
                },
                path: ['size', 'font', '20'],
            },
            22: {
                value: '1.375rem',
                target: 'mobile',
                comment: '22 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '1.375rem',
                    target: 'mobile',
                    comment: '22 pixels',
                },
                name: 'SizeFont22',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '22',
                },
                path: ['size', 'font', '22'],
            },
            24: {
                value: '1.5rem',
                target: 'mobile',
                comment: '24 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '1.5rem',
                    target: 'mobile',
                    comment: '24 pixels',
                },
                name: 'SizeFont24',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '24',
                },
                path: ['size', 'font', '24'],
            },
            26: {
                value: '1.625rem',
                target: 'mobile',
                comment: '26 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '1.625rem',
                    target: 'mobile',
                    comment: '26 pixels',
                },
                name: 'SizeFont26',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '26',
                },
                path: ['size', 'font', '26'],
            },
            28: {
                value: '1.75rem',
                target: 'mobile',
                comment: '28 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '1.75rem',
                    target: 'mobile',
                    comment: '28 pixels',
                },
                name: 'SizeFont28',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '28',
                },
                path: ['size', 'font', '28'],
            },
            30: {
                value: '1.875rem',
                target: 'mobile',
                comment: '30 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '1.875rem',
                    target: 'mobile',
                    comment: '30 pixels',
                },
                name: 'SizeFont30',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '30',
                },
                path: ['size', 'font', '30'],
            },
            32: {
                value: '2rem',
                target: 'mobile',
                comment: '32 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '2rem',
                    target: 'mobile',
                    comment: '32 pixels',
                },
                name: 'SizeFont32',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '32',
                },
                path: ['size', 'font', '32'],
            },
            33: {
                value: '2.0625rem',
                target: 'mobile',
                comment: '33 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '2.0625rem',
                    target: 'mobile',
                    comment: '33 pixels',
                },
                name: 'SizeFont33',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '33',
                },
                path: ['size', 'font', '33'],
            },
            36: {
                value: '2.25rem',
                target: 'mobile',
                comment: '36 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '2.25rem',
                    target: 'mobile',
                    comment: '36 pixels',
                },
                name: 'SizeFont36',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '36',
                },
                path: ['size', 'font', '36'],
            },
            38: {
                value: '2.375rem',
                target: 'mobile',
                comment: '38 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '2.375rem',
                    target: 'mobile',
                    comment: '38 pixels',
                },
                name: 'SizeFont38',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '38',
                },
                path: ['size', 'font', '38'],
            },
            42: {
                value: '2.625rem',
                target: 'mobile',
                comment: '42 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '2.625rem',
                    target: 'mobile',
                    comment: '42 pixels',
                },
                name: 'SizeFont42',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '42',
                },
                path: ['size', 'font', '42'],
            },
            48: {
                value: '3rem',
                target: 'mobile',
                comment: '48 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '3rem',
                    target: 'mobile',
                    comment: '48 pixels',
                },
                name: 'SizeFont48',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '48',
                },
                path: ['size', 'font', '48'],
            },
            54: {
                value: '3.375rem',
                target: 'mobile',
                comment: '54 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '3.375rem',
                    target: 'mobile',
                    comment: '54 pixels',
                },
                name: 'SizeFont54',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '54',
                },
                path: ['size', 'font', '54'],
            },
            56: {
                value: '3.5rem',
                target: 'mobile',
                comment: '56 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '3.5rem',
                    target: 'mobile',
                    comment: '56 pixels',
                },
                name: 'SizeFont56',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '56',
                },
                path: ['size', 'font', '56'],
            },
            66: {
                value: '4.125rem',
                target: 'mobile',
                comment: '66 pixels',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '4.125rem',
                    target: 'mobile',
                    comment: '66 pixels',
                },
                name: 'SizeFont66',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '66',
                },
                path: ['size', 'font', '66'],
            },
            '14px': {
                value: 14,
                target: 'web',
                comment: '0.875rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '14px',
                    target: 'web',
                    comment: '0.875rem',
                },
                name: 'SizeFont14px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '14px',
                },
                path: ['size', 'font', '14px'],
            },
            '15px': {
                value: 15,
                target: 'web',
                comment: '0.9375rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '15px',
                    target: 'web',
                    comment: '0.9375rem',
                },
                name: 'SizeFont15px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '15px',
                },
                path: ['size', 'font', '15px'],
            },
            '16px': {
                value: 16,
                target: 'web',
                comment: '1rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '16px',
                    target: 'web',
                    comment: '1rem',
                },
                name: 'SizeFont16px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '16px',
                },
                path: ['size', 'font', '16px'],
            },
            '18px': {
                value: 18,
                target: 'web',
                comment: '1.125rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '18px',
                    target: 'web',
                    comment: '1.125rem',
                },
                name: 'SizeFont18px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '18px',
                },
                path: ['size', 'font', '18px'],
            },
            '20px': {
                value: 20,
                target: 'web',
                comment: '1.25rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '20px',
                    target: 'web',
                    comment: '1.25rem',
                },
                name: 'SizeFont20px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '20px',
                },
                path: ['size', 'font', '20px'],
            },
            '22px': {
                value: 22,
                target: 'web',
                comment: '1.375rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '22px',
                    target: 'web',
                    comment: '1.375rem',
                },
                name: 'SizeFont22px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '22px',
                },
                path: ['size', 'font', '22px'],
            },
            '24px': {
                value: 24,
                target: 'web',
                comment: '1.5rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '24px',
                    target: 'web',
                    comment: '1.5rem',
                },
                name: 'SizeFont24px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '24px',
                },
                path: ['size', 'font', '24px'],
            },
            '26px': {
                value: 26,
                target: 'web',
                comment: '1.625rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '26px',
                    target: 'web',
                    comment: '1.625rem',
                },
                name: 'SizeFont26px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '26px',
                },
                path: ['size', 'font', '26px'],
            },
            '28px': {
                value: 28,
                target: 'web',
                comment: '1.75rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '28px',
                    target: 'web',
                    comment: '1.75rem',
                },
                name: 'SizeFont28px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '28px',
                },
                path: ['size', 'font', '28px'],
            },
            '30px': {
                value: 30,
                target: 'web',
                comment: '1.875rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '30px',
                    target: 'web',
                    comment: '1.875rem',
                },
                name: 'SizeFont30px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '30px',
                },
                path: ['size', 'font', '30px'],
            },
            '32px': {
                value: 32,
                target: 'web',
                comment: '2rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '32px',
                    target: 'web',
                    comment: '2rem',
                },
                name: 'SizeFont32px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '32px',
                },
                path: ['size', 'font', '32px'],
            },
            '33px': {
                value: 33,
                target: 'web',
                comment: '2.0625rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '33px',
                    target: 'web',
                    comment: '2.0625rem',
                },
                name: 'SizeFont33px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '33px',
                },
                path: ['size', 'font', '33px'],
            },
            '36px': {
                value: 36,
                target: 'web',
                comment: '2.25rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '36px',
                    target: 'web',
                    comment: '2.25rem',
                },
                name: 'SizeFont36px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '36px',
                },
                path: ['size', 'font', '36px'],
            },
            '38px': {
                value: 38,
                target: 'web',
                comment: '2.375rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '38px',
                    target: 'web',
                    comment: '2.375rem',
                },
                name: 'SizeFont38px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '38px',
                },
                path: ['size', 'font', '38px'],
            },
            '42px': {
                value: 42,
                target: 'web',
                comment: '2.625rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '42px',
                    target: 'web',
                    comment: '2.625rem',
                },
                name: 'SizeFont42px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '42px',
                },
                path: ['size', 'font', '42px'],
            },
            '48px': {
                value: 48,
                target: 'web',
                comment: '3rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '48px',
                    target: 'web',
                    comment: '3rem',
                },
                name: 'SizeFont48px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '48px',
                },
                path: ['size', 'font', '48px'],
            },
            '54px': {
                value: 54,
                target: 'web',
                comment: '3.375rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '54px',
                    target: 'web',
                    comment: '3.375rem',
                },
                name: 'SizeFont54px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '54px',
                },
                path: ['size', 'font', '54px'],
            },
            '56px': {
                value: 56,
                target: 'web',
                comment: '3.5rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '56px',
                    target: 'web',
                    comment: '3.5rem',
                },
                name: 'SizeFont56px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '56px',
                },
                path: ['size', 'font', '56px'],
            },
            '66px': {
                value: 66,
                target: 'web',
                comment: '4.125rem',
                filePath: './src/tokens/font-size.json',
                isSource: true,
                original: {
                    value: '66px',
                    target: 'web',
                    comment: '4.125rem',
                },
                name: 'SizeFont66px',
                attributes: {
                    category: 'size',
                    type: 'font',
                    item: '66px',
                },
                path: ['size', 'font', '66px'],
            },
        },
        'line-height': {
            24: {
                value: '1.5rem',
                target: 'mobile',
                comment: '24px',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '1.5rem',
                    target: 'mobile',
                    comment: '24px',
                },
                name: 'SizeLineHeight24',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '24',
                },
                path: ['size', 'line-height', '24'],
            },
            28: {
                value: '1.75rem',
                target: 'mobile',
                comment: '28px',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '1.75rem',
                    target: 'mobile',
                    comment: '28px',
                },
                name: 'SizeLineHeight28',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '28',
                },
                path: ['size', 'line-height', '28'],
            },
            32: {
                value: '2rem',
                target: 'mobile',
                comment: '32px',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '2rem',
                    target: 'mobile',
                    comment: '32px',
                },
                name: 'SizeLineHeight32',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '32',
                },
                path: ['size', 'line-height', '32'],
            },
            40: {
                value: '2.5rem',
                target: 'mobile',
                comment: '40px',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '2.5rem',
                    target: 'mobile',
                    comment: '40px',
                },
                name: 'SizeLineHeight40',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '40',
                },
                path: ['size', 'line-height', '40'],
            },
            42: {
                value: '2.625rem',
                target: 'mobile',
                comment: '42px',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '2.625rem',
                    target: 'mobile',
                    comment: '42px',
                },
                name: 'SizeLineHeight42',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '42',
                },
                path: ['size', 'line-height', '42'],
            },
            44: {
                value: '2.75rem',
                target: 'mobile',
                comment: '44px',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '2.75rem',
                    target: 'mobile',
                    comment: '44px',
                },
                name: 'SizeLineHeight44',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '44',
                },
                path: ['size', 'line-height', '44'],
            },
            48: {
                value: '3rem',
                target: 'mobile',
                comment: '48px',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '3rem',
                    target: 'mobile',
                    comment: '48px',
                },
                name: 'SizeLineHeight48',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '48',
                },
                path: ['size', 'line-height', '48'],
            },
            52: {
                value: '3.25rem',
                target: 'mobile',
                comment: '52px',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '3.25rem',
                    target: 'mobile',
                    comment: '52px',
                },
                name: 'SizeLineHeight52',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '52',
                },
                path: ['size', 'line-height', '52'],
            },
            56: {
                value: '3.5rem',
                target: 'mobile',
                comment: '56px',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '3.5rem',
                    target: 'mobile',
                    comment: '56px',
                },
                name: 'SizeLineHeight56',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '56',
                },
                path: ['size', 'line-height', '56'],
            },
            64: {
                value: '4rem',
                target: 'mobile',
                comment: '64px',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '4rem',
                    target: 'mobile',
                    comment: '64px',
                },
                name: 'SizeLineHeight64',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '64',
                },
                path: ['size', 'line-height', '64'],
            },
            72: {
                value: '4.5rem',
                target: 'mobile',
                comment: '72px',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '4.5rem',
                    target: 'mobile',
                    comment: '72px',
                },
                name: 'SizeLineHeight72',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '72',
                },
                path: ['size', 'line-height', '72'],
            },
            '24px': {
                value: 24,
                target: 'web',
                comment: '1.5rem',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '24px',
                    target: 'web',
                    comment: '1.5rem',
                },
                name: 'SizeLineHeight24px',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '24px',
                },
                path: ['size', 'line-height', '24px'],
            },
            '28px': {
                value: 28,
                target: 'web',
                comment: '1.75rem',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '28px',
                    target: 'web',
                    comment: '1.75rem',
                },
                name: 'SizeLineHeight28px',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '28px',
                },
                path: ['size', 'line-height', '28px'],
            },
            '32px': {
                value: 32,
                target: 'web',
                comment: '2rem',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '32px',
                    target: 'web',
                    comment: '2rem',
                },
                name: 'SizeLineHeight32px',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '32px',
                },
                path: ['size', 'line-height', '32px'],
            },
            '40px': {
                value: 40,
                target: 'web',
                comment: '2.5rem',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '40px',
                    target: 'web',
                    comment: '2.5rem',
                },
                name: 'SizeLineHeight40px',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '40px',
                },
                path: ['size', 'line-height', '40px'],
            },
            '42px': {
                value: 42,
                target: 'web',
                comment: '2.625rem',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '42px',
                    target: 'web',
                    comment: '2.625rem',
                },
                name: 'SizeLineHeight42px',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '42px',
                },
                path: ['size', 'line-height', '42px'],
            },
            '44px': {
                value: 44,
                target: 'web',
                comment: '2.75rem',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '44px',
                    target: 'web',
                    comment: '2.75rem',
                },
                name: 'SizeLineHeight44px',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '44px',
                },
                path: ['size', 'line-height', '44px'],
            },
            '48px': {
                value: 48,
                target: 'web',
                comment: '3rem',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '48px',
                    target: 'web',
                    comment: '3rem',
                },
                name: 'SizeLineHeight48px',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '48px',
                },
                path: ['size', 'line-height', '48px'],
            },
            '52px': {
                value: 52,
                target: 'web',
                comment: '3.25rem',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '52px',
                    target: 'web',
                    comment: '3.25rem',
                },
                name: 'SizeLineHeight52px',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '52px',
                },
                path: ['size', 'line-height', '52px'],
            },
            '56px': {
                value: 56,
                target: 'web',
                comment: '3.5rem',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '56px',
                    target: 'web',
                    comment: '3.5rem',
                },
                name: 'SizeLineHeight56px',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '56px',
                },
                path: ['size', 'line-height', '56px'],
            },
            '64px': {
                value: 64,
                target: 'web',
                comment: '4rem',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '64px',
                    target: 'web',
                    comment: '4rem',
                },
                name: 'SizeLineHeight64px',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '64px',
                },
                path: ['size', 'line-height', '64px'],
            },
            '72px': {
                value: 72,
                target: 'web',
                comment: '4.5rem',
                filePath: './src/tokens/line-height.json',
                isSource: true,
                original: {
                    value: '72px',
                    target: 'web',
                    comment: '4.5rem',
                },
                name: 'SizeLineHeight72px',
                attributes: {
                    category: 'size',
                    type: 'line-height',
                    item: '72px',
                },
                path: ['size', 'line-height', '72px'],
            },
        },
    },
    font: {
        weight: {
            36: {
                value: 36,
                variable: true,
                'font-family': '"If Sans", Arial, sans-serif',
                filePath: './src/tokens/font-weight.json',
                isSource: true,
                original: {
                    value: 36,
                    variable: true,
                    'font-family': '"If Sans", Arial, sans-serif',
                },
                name: 'FontWeight36',
                attributes: {
                    category: 'font',
                    type: 'weight',
                    item: '36',
                },
                path: ['font', 'weight', '36'],
            },
            40: {
                value: 40,
                variable: true,
                'font-family': '"If Sans", Arial, sans-serif',
                filePath: './src/tokens/font-weight.json',
                isSource: true,
                original: {
                    value: 40,
                    variable: true,
                    'font-family': '"If Sans", Arial, sans-serif',
                },
                name: 'FontWeight40',
                attributes: {
                    category: 'font',
                    type: 'weight',
                    item: '40',
                },
                path: ['font', 'weight', '40'],
            },
            45: {
                value: 45,
                variable: true,
                'font-family': '"If Sans", Arial, sans-serif',
                filePath: './src/tokens/font-weight.json',
                isSource: true,
                original: {
                    value: 45,
                    variable: true,
                    'font-family': '"If Sans", Arial, sans-serif',
                },
                name: 'FontWeight45',
                attributes: {
                    category: 'font',
                    type: 'weight',
                    item: '45',
                },
                path: ['font', 'weight', '45'],
            },
            48: {
                value: 48,
                variable: true,
                'font-family': '"If Sans", Arial, sans-serif',
                filePath: './src/tokens/font-weight.json',
                isSource: true,
                original: {
                    value: 48,
                    variable: true,
                    'font-family': '"If Sans", Arial, sans-serif',
                },
                name: 'FontWeight48',
                attributes: {
                    category: 'font',
                    type: 'weight',
                    item: '48',
                },
                path: ['font', 'weight', '48'],
            },
            50: {
                value: 50,
                variable: true,
                'font-family': '"If Sans", Arial, sans-serif',
                filePath: './src/tokens/font-weight.json',
                isSource: true,
                original: {
                    value: 50,
                    variable: true,
                    'font-family': '"If Sans", Arial, sans-serif',
                },
                name: 'FontWeight50',
                attributes: {
                    category: 'font',
                    type: 'weight',
                    item: '50',
                },
                path: ['font', 'weight', '50'],
            },
            54: {
                value: 54,
                variable: true,
                'font-family': '"If Sans", Arial, sans-serif',
                filePath: './src/tokens/font-weight.json',
                isSource: true,
                original: {
                    value: 54,
                    variable: true,
                    'font-family': '"If Sans", Arial, sans-serif',
                },
                name: 'FontWeight54',
                attributes: {
                    category: 'font',
                    type: 'weight',
                    item: '54',
                },
                path: ['font', 'weight', '54'],
            },
            64: {
                value: 64,
                variable: true,
                'font-family': '"If Sans", Arial, sans-serif',
                filePath: './src/tokens/font-weight.json',
                isSource: true,
                original: {
                    value: 64,
                    variable: true,
                    'font-family': '"If Sans", Arial, sans-serif',
                },
                name: 'FontWeight64',
                attributes: {
                    category: 'font',
                    type: 'weight',
                    item: '64',
                },
                path: ['font', 'weight', '64'],
            },
            70: {
                value: 70,
                variable: true,
                'font-family': '"If Sans", Arial, sans-serif',
                filePath: './src/tokens/font-weight.json',
                isSource: true,
                original: {
                    value: 70,
                    variable: true,
                    'font-family': '"If Sans", Arial, sans-serif',
                },
                name: 'FontWeight70',
                attributes: {
                    category: 'font',
                    type: 'weight',
                    item: '70',
                },
                path: ['font', 'weight', '70'],
            },
            78: {
                value: 78,
                variable: true,
                'font-family': '"If Sans", Arial, sans-serif',
                filePath: './src/tokens/font-weight.json',
                isSource: true,
                original: {
                    value: 78,
                    variable: true,
                    'font-family': '"If Sans", Arial, sans-serif',
                },
                name: 'FontWeight78',
                attributes: {
                    category: 'font',
                    type: 'weight',
                    item: '78',
                },
                path: ['font', 'weight', '78'],
            },
            82: {
                value: 82,
                variable: true,
                'font-family': '"If Sans", Arial, sans-serif',
                filePath: './src/tokens/font-weight.json',
                isSource: true,
                original: {
                    value: 82,
                    variable: true,
                    'font-family': '"If Sans", Arial, sans-serif',
                },
                name: 'FontWeight82',
                attributes: {
                    category: 'font',
                    type: 'weight',
                    item: '82',
                },
                path: ['font', 'weight', '82'],
            },
            85: {
                value: 85,
                variable: true,
                'font-family': '"If Sans", Arial, sans-serif',
                filePath: './src/tokens/font-weight.json',
                isSource: true,
                original: {
                    value: 85,
                    variable: true,
                    'font-family': '"If Sans", Arial, sans-serif',
                },
                name: 'FontWeight85',
                attributes: {
                    category: 'font',
                    type: 'weight',
                    item: '85',
                },
                path: ['font', 'weight', '85'],
            },
            104: {
                value: 104,
                variable: true,
                'font-family': '"If Sans", Arial, sans-serif',
                filePath: './src/tokens/font-weight.json',
                isSource: true,
                original: {
                    value: 104,
                    variable: true,
                    'font-family': '"If Sans", Arial, sans-serif',
                },
                name: 'FontWeight104',
                attributes: {
                    category: 'font',
                    type: 'weight',
                    item: '104',
                },
                path: ['font', 'weight', '104'],
            },
            126: {
                value: 126,
                variable: true,
                'font-family': '"If Sans", Arial, sans-serif',
                filePath: './src/tokens/font-weight.json',
                isSource: true,
                original: {
                    value: 126,
                    variable: true,
                    'font-family': '"If Sans", Arial, sans-serif',
                },
                name: 'FontWeight126',
                attributes: {
                    category: 'font',
                    type: 'weight',
                    item: '126',
                },
                path: ['font', 'weight', '126'],
            },
        },
        family: {
            'sans-thin': {
                value: 'If Sans Thin',
                filePath: './src/tokens/font.json',
                isSource: true,
                original: {
                    value: '{asset.font.ifsans-thin.name.value}',
                },
                name: 'FontFamilySansThin',
                attributes: {
                    category: 'font',
                    type: 'family',
                    item: 'sans-thin',
                },
                path: ['font', 'family', 'sans-thin'],
            },
            'sans-light': {
                value: 'If Sans Light',
                filePath: './src/tokens/font.json',
                isSource: true,
                original: {
                    value: '{asset.font.ifsans-light.name.value}',
                },
                name: 'FontFamilySansLight',
                attributes: {
                    category: 'font',
                    type: 'family',
                    item: 'sans-light',
                },
                path: ['font', 'family', 'sans-light'],
            },
            sans: {
                value: 'If Sans',
                filePath: './src/tokens/font.json',
                isSource: true,
                original: {
                    value: '{asset.font.ifsans-regular.name.value}',
                },
                name: 'FontFamilySans',
                attributes: {
                    category: 'font',
                    type: 'family',
                    item: 'sans',
                },
                path: ['font', 'family', 'sans'],
            },
            'sans-medium': {
                value: 'If Sans Medium',
                filePath: './src/tokens/font.json',
                isSource: true,
                original: {
                    value: '{asset.font.ifsans-medium.name.value}',
                },
                name: 'FontFamilySansMedium',
                attributes: {
                    category: 'font',
                    type: 'family',
                    item: 'sans-medium',
                },
                path: ['font', 'family', 'sans-medium'],
            },
            'sans-bold': {
                value: 'If Sans Bold',
                filePath: './src/tokens/font.json',
                isSource: true,
                original: {
                    value: '{asset.font.ifsans-bold.name.value}',
                },
                name: 'FontFamilySansBold',
                attributes: {
                    category: 'font',
                    type: 'family',
                    item: 'sans-bold',
                },
                path: ['font', 'family', 'sans-bold'],
            },
            'sans-thin-italic': {
                value: 'If Sans Thin Italic',
                filePath: './src/tokens/font.json',
                isSource: true,
                original: {
                    value: '{asset.font.ifsans-thin-italic.name.value}',
                },
                name: 'FontFamilySansThinItalic',
                attributes: {
                    category: 'font',
                    type: 'family',
                    item: 'sans-thin-italic',
                },
                path: ['font', 'family', 'sans-thin-italic'],
            },
            'sans-light-italic': {
                value: 'If Sans Light Italic',
                filePath: './src/tokens/font.json',
                isSource: true,
                original: {
                    value: '{asset.font.ifsans-light-italic.name.value}',
                },
                name: 'FontFamilySansLightItalic',
                attributes: {
                    category: 'font',
                    type: 'family',
                    item: 'sans-light-italic',
                },
                path: ['font', 'family', 'sans-light-italic'],
            },
            'sans-italic': {
                value: 'If Sans Italic',
                filePath: './src/tokens/font.json',
                isSource: true,
                original: {
                    value: '{asset.font.ifsans-regular-italic.name.value}',
                },
                name: 'FontFamilySansItalic',
                attributes: {
                    category: 'font',
                    type: 'family',
                    item: 'sans-italic',
                },
                path: ['font', 'family', 'sans-italic'],
            },
            'sans-medium-italic': {
                value: 'If Sans Medium Italic',
                filePath: './src/tokens/font.json',
                isSource: true,
                original: {
                    value: '{asset.font.ifsans-medium-italic.name.value}',
                },
                name: 'FontFamilySansMediumItalic',
                attributes: {
                    category: 'font',
                    type: 'family',
                    item: 'sans-medium-italic',
                },
                path: ['font', 'family', 'sans-medium-italic'],
            },
            'sans-bold-italic': {
                value: 'If Sans Bold Italic',
                filePath: './src/tokens/font.json',
                isSource: true,
                original: {
                    value: '{asset.font.ifsans-bold-italic.name.value}',
                },
                name: 'FontFamilySansBoldItalic',
                attributes: {
                    category: 'font',
                    type: 'family',
                    item: 'sans-bold-italic',
                },
                path: ['font', 'family', 'sans-bold-italic'],
            },
        },
    },
};

(function (root, factory) {
    if (typeof module === 'object' && module.exports) {
        module.exports = factory();
    } else if (typeof exports === 'object') {
        exports['IDS_TYPOGRAPHY'] = factory();
    } else if (typeof define === 'function' && define.amd) {
        define([], factory);
    } else {
        root['IDS_TYPOGRAPHY'] = factory();
    }
})(this, function () {
    return { ...deprecated_variables, ...generated_variables };
});
