// @ids-core/utils/src/variables/js/deprecated-variables.esm.js
/*
 * Deprecated variables
 * @todo remove in version 16
 */

export const DepthOpacityHidden = '0'; // Makes the element completely tranparent.

export const DepthOpacityBorder = '0.2'; // Gives 20% opacity to an element.

export const DepthOpacityDisabled = '0.3'; // Gives 30% opacity to an element.

export const DepthOpacityBackdrop = '0.4'; // Gives 40% opacity to an element.

export const DepthOpacityVisible = '1'; // Makes an element completely visible.

export const DepthShadowLight100 = '0 8px 4px -2px rgba(110,98,94, 0.08), 0 8px 8px rgba(110,98,94, 0.12)';

export const DepthShadowLight200 = '0 8px 4px -2px rgba(110,98,94, 0.08), 0 12px 16px rgba(110,98,94, 0.16)';

export const DepthShadowLight300 = '0 8px 4px -2px rgba(110,98,94, 0.08), 0 16px 24px rgba(110,98,94, 0.20)';

export const DepthShadowLight400 = '0 8px 4px -2px rgba(110,98,94, 0.08), 0 20px 32px rgba(110,98,94, 0.24)';

export const DepthShadowDark100 = '0 8px 4px -2px rgba(51,30,17, 0.08), 0 8px 8px rgba(51,30,17, 0.20)';

export const DepthShadowDark200 = '0 8px 4px -2px rgba(51,30,17, 0.08), 0 12px 16px rgba(51,30,17, 0.24)';

export const DepthShadowDark300 = '0 8px 4px -2px rgba(51,30,17, 0.08), 0 16px 24px rgba(51,30,17, 0.28)';

export const DepthShadowDark400 = '0 8px 4px -2px rgba(51,30,17, 0.08), 0 20px 32px rgba(51,30,17, 0.32)';

export const DepthZIndexDeep = '-999999'; // Deep z-index is used to stack something behind everything else.

export const DepthZIndexDefault = '1'; // The default z-index for components and elements inside components.

export const DepthZIndexMasked = '100'; // Default z-index for masked interface elements.

export const DepthZIndexMask = '200'; // Default z-index for masking interface elements.

export const DepthZIndexSticky = '300'; // Default z-index for sticky interface elements.

export const DepthZIndexGlobalHeader = '400'; // Default z-index for the Global Header.

export const DepthZIndexToast = '500'; // Default z-index for toast messages.

export const DepthZIndexDropdown = '600'; // Default z-index for dropdowns which makes sure the dropdown stacks above toasts and sticky elements.

export const DepthZIndexOverlay = '700'; // Default z-index for modal and popup overlays.

export const DepthZIndexLoader = '800'; // Default z-index for loaders, stacking on top of overlay.

export const DepthZIndexModal = '900'; // Default z-index for modals that stacks on top of overlays and other elements, but still allows popovers to be visible.

export const DepthZIndexPopover = '950'; // Default z-index for popover that stacks on top of all other elements.

export const Size2 = '0.125rem'; // 2px

export const Size4 = '0.25rem'; // 4px

export const Size8 = '0.5rem'; // 8px

export const Size12 = '0.75rem'; // 12px

export const Size16 = '1rem'; // 16px

export const Size20 = '1.25rem'; // 20px

export const Size24 = '1.5rem'; // 24px

export const Size28 = '1.75rem'; // 28px

export const Size32 = '2rem'; // 32px

export const Size36 = '2.25rem'; // 36px

export const Size40 = '2.5rem'; // 40px

export const Size44 = '2.75rem'; // 44px

export const Size48 = '3rem'; // 48px

export const Size52 = '3.25rem'; // 52px

export const Size56 = '3.5rem'; // 56px

export const Size60 = '3.75rem'; // 60px

export const Size64 = '4rem'; // 64px

export const Size68 = '4.25rem'; // 68px

export const Size72 = '4.5rem'; // 72px

export const Size76 = '4.75rem'; // 76px

export const Size80 = '5rem'; // 80px

export const Size84 = '5.25rem'; // 84px

export const Size88 = '5.5rem'; // 88px

export const Size92 = '5.75rem'; // 92px

export const Size96 = '6rem'; // 96px

export const Size100 = '6.25rem'; // 100px

export const Size104 = '6.5rem'; // 104px

export const Size120 = '7.5rem'; // 120px

export const Size2px = '2px'; // 0.125rem

export const Size4px = '4px'; // 0.25rem

export const Size8px = '8px'; // 0.5rem

export const Size12px = '12px'; // 0.75rem

export const Size16px = '16px'; // 1rem

export const Size20px = '20px'; // 1.25rem

export const Size24px = '24px'; // 1.5rem

export const Size28px = '28px'; // 1.75rem

export const Size32px = '32px'; // 2rem

export const Size36px = '36px'; // 2.25rem

export const Size40px = '40px'; // 2.5rem

export const Size44px = '44px'; // 2.75rem

export const Size48px = '48px'; // 3rem

export const Size52px = '52px'; // 3.25rem

export const Size56px = '56px'; // 3.5rem

export const Size60px = '60px'; // 3.75rem

export const Size64px = '64px'; // 4rem

export const Size68px = '68px'; // 4.25rem

export const Size72px = '72px'; // 4.5rem

export const Size76px = '76px'; // 4.75rem

export const Size80px = '80px'; // 5rem

export const Size84px = '84px'; // 5.25rem

export const Size88px = '88px'; // 5.5rem

export const Size92px = '92px'; // 5.75rem

export const Size96px = '96px'; // 6rem

export const Size100px = '100px'; // 6.25rem

export const Size104px = '104px'; // 6.5rem

export const Size120px = '120px'; // 7.5rem

export const SizeSpacing2 = '0.125rem';

export const SizeSpacing4 = '0.25rem';

export const SizeSpacing8 = '0.5rem';

export const SizeSpacing12 = '0.75rem';

export const SizeSpacing16 = '1rem';

export const SizeSpacing24 = '1.5rem';

export const SizeSpacing32 = '2rem';

export const SizeSpacing40 = '2.5rem';

export const SizeSpacing48 = '3rem';

export const SizeSpacing56 = '3.5rem';

export const SizeSpacing64 = '4rem';

export const SizeSpacing88 = '5.5rem';

export const SizeSpacing120 = '7.5rem';

export const SizeSpacing2px = '2px';

export const SizeSpacing4px = '4px';

export const SizeSpacing8px = '8px';

export const SizeSpacing12px = '12px';

export const SizeSpacing16px = '16px';

export const SizeSpacing24px = '24px';

export const SizeSpacing32px = '32px';

export const SizeSpacing40px = '40px';

export const SizeSpacing48px = '48px';

export const SizeSpacing56px = '56px';

export const SizeSpacing64px = '64px';

export const SizeSpacing88px = '88px';

export const SizeSpacing120px = '120px';
