// @ids-core/utils/src/variables/js/deprecated-variables.module.js
/*
 * Deprecated variables
 * @todo remove in version 16
 */
module.exports = {
  depth: {
    opacity: {
      hidden: {
        value: '0',
        target: 'web',
        comment: 'Makes the element completely tranparent.',
        filePath: './src/tokens/opacity.json',
        isSource: true,
        original: {
          value: '0',
          target: 'web',
          comment: 'Makes the element completely tranparent.'
        },
        name: 'DepthOpacityHidden',
        attributes: {
          category: 'depth',
          type: 'opacity',
          item: 'hidden'
        },
        path: ['depth', 'opacity', 'hidden']
      },
      border: {
        value: '0.2',
        target: 'web',
        comment: 'Gives 20% opacity to an element.',
        filePath: './src/tokens/opacity.json',
        isSource: true,
        original: {
          value: '0.2',
          target: 'web',
          comment: 'Gives 20% opacity to an element.'
        },
        name: 'DepthOpacityBorder',
        attributes: {
          category: 'depth',
          type: 'opacity',
          item: 'border'
        },
        path: ['depth', 'opacity', 'border']
      },
      disabled: {
        value: '0.3',
        target: 'web',
        comment: 'Gives 30% opacity to an element.',
        filePath: './src/tokens/opacity.json',
        isSource: true,
        original: {
          value: '0.3',
          target: 'web',
          comment: 'Gives 30% opacity to an element.'
        },
        name: 'DepthOpacityDisabled',
        attributes: {
          category: 'depth',
          type: 'opacity',
          item: 'disabled'
        },
        path: ['depth', 'opacity', 'disabled']
      },
      backdrop: {
        value: '0.4',
        target: 'web',
        comment: 'Gives 40% opacity to an element.',
        filePath: './src/tokens/opacity.json',
        isSource: true,
        original: {
          value: '0.4',
          target: 'web',
          comment: 'Gives 40% opacity to an element.'
        },
        name: 'DepthOpacityBackdrop',
        attributes: {
          category: 'depth',
          type: 'opacity',
          item: 'backdrop'
        },
        path: ['depth', 'opacity', 'backdrop']
      },
      visible: {
        value: '1',
        target: 'web',
        comment: 'Makes an element completely visible.',
        filePath: './src/tokens/opacity.json',
        isSource: true,
        original: {
          value: '1',
          target: 'web',
          comment: 'Makes an element completely visible.'
        },
        name: 'DepthOpacityVisible',
        attributes: {
          category: 'depth',
          type: 'opacity',
          item: 'visible'
        },
        path: ['depth', 'opacity', 'visible']
      }
    },
    shadow: {
      light: {
        100: {
          value: '0 8px 4px -2px rgba(110,98,94, 0.08), 0 8px 8px rgba(110,98,94, 0.12)',
          color: '#6e625e',
          colorName: 'LB 1, LIGHT BROWN',
          opacity: '12%',
          offset: {
            x: 0,
            y: '8px'
          },
          blur: '8px',
          target: 'web',
          filePath: './src/tokens/shadow.json',
          isSource: true,
          original: {
            value: '0 8px 4px -2px rgba(110,98,94, 0.08), 0 8px 8px rgba(110,98,94, 0.12)',
            color: '#6e625e',
            colorName: 'LB 1, LIGHT BROWN',
            opacity: '12%',
            offset: {
              x: 0,
              y: '8px'
            },
            blur: '8px',
            target: 'web'
          },
          name: 'DepthShadowLight100',
          attributes: {
            category: 'depth',
            type: 'shadow',
            item: 'light',
            subitem: '100'
          },
          path: ['depth', 'shadow', 'light', '100']
        },
        200: {
          value: '0 8px 4px -2px rgba(110,98,94, 0.08), 0 12px 16px rgba(110,98,94, 0.16)',
          color: '#6e625e',
          colorName: 'LB 1, LIGHT BROWN',
          opacity: '16%',
          offset: {
            x: 0,
            y: '12px'
          },
          blur: '16px',
          target: 'web',
          filePath: './src/tokens/shadow.json',
          isSource: true,
          original: {
            value: '0 8px 4px -2px rgba(110,98,94, 0.08), 0 12px 16px rgba(110,98,94, 0.16)',
            color: '#6e625e',
            colorName: 'LB 1, LIGHT BROWN',
            opacity: '16%',
            offset: {
              x: 0,
              y: '12px'
            },
            blur: '16px',
            target: 'web'
          },
          name: 'DepthShadowLight200',
          attributes: {
            category: 'depth',
            type: 'shadow',
            item: 'light',
            subitem: '200'
          },
          path: ['depth', 'shadow', 'light', '200']
        },
        300: {
          value: '0 8px 4px -2px rgba(110,98,94, 0.08), 0 16px 24px rgba(110,98,94, 0.20)',
          color: '#6e625e',
          colorName: 'LB 1, LIGHT BROWN',
          opacity: '20%',
          offset: {
            x: 0,
            y: '16px'
          },
          blur: '24px',
          target: 'web',
          filePath: './src/tokens/shadow.json',
          isSource: true,
          original: {
            value: '0 8px 4px -2px rgba(110,98,94, 0.08), 0 16px 24px rgba(110,98,94, 0.20)',
            color: '#6e625e',
            colorName: 'LB 1, LIGHT BROWN',
            opacity: '20%',
            offset: {
              x: 0,
              y: '16px'
            },
            blur: '24px',
            target: 'web'
          },
          name: 'DepthShadowLight300',
          attributes: {
            category: 'depth',
            type: 'shadow',
            item: 'light',
            subitem: '300'
          },
          path: ['depth', 'shadow', 'light', '300']
        },
        400: {
          value: '0 8px 4px -2px rgba(110,98,94, 0.08), 0 20px 32px rgba(110,98,94, 0.24)',
          color: '#6e625e',
          colorName: 'LB 1, LIGHT BROWN',
          opacity: '24%',
          offset: {
            x: 0,
            y: '20px'
          },
          blur: '32px',
          target: 'web',
          filePath: './src/tokens/shadow.json',
          isSource: true,
          original: {
            value: '0 8px 4px -2px rgba(110,98,94, 0.08), 0 20px 32px rgba(110,98,94, 0.24)',
            color: '#6e625e',
            colorName: 'LB 1, LIGHT BROWN',
            opacity: '24%',
            offset: {
              x: 0,
              y: '20px'
            },
            blur: '32px',
            target: 'web'
          },
          name: 'DepthShadowLight400',
          attributes: {
            category: 'depth',
            type: 'shadow',
            item: 'light',
            subitem: '400'
          },
          path: ['depth', 'shadow', 'light', '400']
        }
      },
      dark: {
        100: {
          value: '0 8px 4px -2px rgba(51,30,17, 0.08), 0 8px 8px rgba(51,30,17, 0.20)',
          color: '#331e11',
          colorName: 'BR 1, DARK BROWN',
          opacity: '20%',
          offset: {
            x: 0,
            y: '8px'
          },
          blur: '8px',
          target: 'web',
          filePath: './src/tokens/shadow.json',
          isSource: true,
          original: {
            value: '0 8px 4px -2px rgba(51,30,17, 0.08), 0 8px 8px rgba(51,30,17, 0.20)',
            color: '#331e11',
            colorName: 'BR 1, DARK BROWN',
            opacity: '20%',
            offset: {
              x: 0,
              y: '8px'
            },
            blur: '8px',
            target: 'web'
          },
          name: 'DepthShadowDark100',
          attributes: {
            category: 'depth',
            type: 'shadow',
            item: 'dark',
            subitem: '100'
          },
          path: ['depth', 'shadow', 'dark', '100']
        },
        200: {
          value: '0 8px 4px -2px rgba(51,30,17, 0.08), 0 12px 16px rgba(51,30,17, 0.24)',
          color: '#331e11',
          colorName: 'BR 1, DARK BROWN',
          opacity: '24%',
          offset: {
            x: 0,
            y: '12px'
          },
          blur: '16px',
          target: 'web',
          filePath: './src/tokens/shadow.json',
          isSource: true,
          original: {
            value: '0 8px 4px -2px rgba(51,30,17, 0.08), 0 12px 16px rgba(51,30,17, 0.24)',
            color: '#331e11',
            colorName: 'BR 1, DARK BROWN',
            opacity: '24%',
            offset: {
              x: 0,
              y: '12px'
            },
            blur: '16px',
            target: 'web'
          },
          name: 'DepthShadowDark200',
          attributes: {
            category: 'depth',
            type: 'shadow',
            item: 'dark',
            subitem: '200'
          },
          path: ['depth', 'shadow', 'dark', '200']
        },
        300: {
          value: '0 8px 4px -2px rgba(51,30,17, 0.08), 0 16px 24px rgba(51,30,17, 0.28)',
          color: '#331e11',
          colorName: 'BR 1, DARK BROWN',
          opacity: '28%',
          offset: {
            x: 0,
            y: '16px'
          },
          blur: '24px',
          target: 'web',
          filePath: './src/tokens/shadow.json',
          isSource: true,
          original: {
            value: '0 8px 4px -2px rgba(51,30,17, 0.08), 0 16px 24px rgba(51,30,17, 0.28)',
            color: '#331e11',
            colorName: 'BR 1, DARK BROWN',
            opacity: '28%',
            offset: {
              x: 0,
              y: '16px'
            },
            blur: '24px',
            target: 'web'
          },
          name: 'DepthShadowDark300',
          attributes: {
            category: 'depth',
            type: 'shadow',
            item: 'dark',
            subitem: '300'
          },
          path: ['depth', 'shadow', 'dark', '300']
        },
        400: {
          value: '0 8px 4px -2px rgba(51,30,17, 0.08), 0 20px 32px rgba(51,30,17, 0.32)',
          color: '#331e11',
          colorName: 'BR 1, DARK BROWN',
          opacity: '32%',
          offset: {
            x: 0,
            y: '20px'
          },
          blur: '32px',
          target: 'web',
          filePath: './src/tokens/shadow.json',
          isSource: true,
          original: {
            value: '0 8px 4px -2px rgba(51,30,17, 0.08), 0 20px 32px rgba(51,30,17, 0.32)',
            color: '#331e11',
            colorName: 'BR 1, DARK BROWN',
            opacity: '32%',
            offset: {
              x: 0,
              y: '20px'
            },
            blur: '32px',
            target: 'web'
          },
          name: 'DepthShadowDark400',
          attributes: {
            category: 'depth',
            type: 'shadow',
            item: 'dark',
            subitem: '400'
          },
          path: ['depth', 'shadow', 'dark', '400']
        }
      }
    },
    'z-index': {
      deep: {
        value: '-999999',
        target: 'web',
        comment: 'Deep z-index is used to stack something behind everything else.',
        filePath: './src/tokens/zindex.json',
        isSource: true,
        original: {
          value: '-999999',
          target: 'web',
          comment: 'Deep z-index is used to stack something behind everything else.'
        },
        name: 'DepthZIndexDeep',
        attributes: {
          category: 'depth',
          type: 'z-index',
          item: 'deep'
        },
        path: ['depth', 'z-index', 'deep']
      },
      default: {
        value: '1',
        target: 'web',
        comment: 'The default z-index for components and elements inside components.',
        filePath: './src/tokens/zindex.json',
        isSource: true,
        original: {
          value: '1',
          target: 'web',
          comment: 'The default z-index for components and elements inside components.'
        },
        name: 'DepthZIndexDefault',
        attributes: {
          category: 'depth',
          type: 'z-index',
          item: 'default'
        },
        path: ['depth', 'z-index', 'default']
      },
      masked: {
        value: '100',
        target: 'web',
        comment: 'Default z-index for masked interface elements.',
        filePath: './src/tokens/zindex.json',
        isSource: true,
        original: {
          value: '100',
          target: 'web',
          comment: 'Default z-index for masked interface elements.'
        },
        name: 'DepthZIndexMasked',
        attributes: {
          category: 'depth',
          type: 'z-index',
          item: 'masked'
        },
        path: ['depth', 'z-index', 'masked']
      },
      mask: {
        value: '200',
        target: 'web',
        comment: 'Default z-index for masking interface elements.',
        filePath: './src/tokens/zindex.json',
        isSource: true,
        original: {
          value: '200',
          target: 'web',
          comment: 'Default z-index for masking interface elements.'
        },
        name: 'DepthZIndexMask',
        attributes: {
          category: 'depth',
          type: 'z-index',
          item: 'mask'
        },
        path: ['depth', 'z-index', 'mask']
      },
      sticky: {
        value: '300',
        target: 'web',
        comment: 'Default z-index for sticky interface elements.',
        filePath: './src/tokens/zindex.json',
        isSource: true,
        original: {
          value: '300',
          target: 'web',
          comment: 'Default z-index for sticky interface elements.'
        },
        name: 'DepthZIndexSticky',
        attributes: {
          category: 'depth',
          type: 'z-index',
          item: 'sticky'
        },
        path: ['depth', 'z-index', 'sticky']
      },
      'global-header': {
        value: '400',
        target: 'web',
        comment: 'Default z-index for the Global Header.',
        filePath: './src/tokens/zindex.json',
        isSource: true,
        original: {
          value: '400',
          target: 'web',
          comment: 'Default z-index for the Global Header.'
        },
        name: 'DepthZIndexGlobalHeader',
        attributes: {
          category: 'depth',
          type: 'z-index',
          item: 'global-header'
        },
        path: ['depth', 'z-index', 'global-header']
      },
      toast: {
        value: '500',
        target: 'web',
        comment: 'Default z-index for toast messages.',
        filePath: './src/tokens/zindex.json',
        isSource: true,
        original: {
          value: '500',
          target: 'web',
          comment: 'Default z-index for toast messages.'
        },
        name: 'DepthZIndexToast',
        attributes: {
          category: 'depth',
          type: 'z-index',
          item: 'toast'
        },
        path: ['depth', 'z-index', 'toast']
      },
      dropdown: {
        value: '600',
        target: 'web',
        comment: 'Default z-index for dropdowns which makes sure the dropdown stacks above toasts and sticky elements.',
        filePath: './src/tokens/zindex.json',
        isSource: true,
        original: {
          value: '600',
          target: 'web',
          comment:
            'Default z-index for dropdowns which makes sure the dropdown stacks above toasts and sticky elements.'
        },
        name: 'DepthZIndexDropdown',
        attributes: {
          category: 'depth',
          type: 'z-index',
          item: 'dropdown'
        },
        path: ['depth', 'z-index', 'dropdown']
      },
      overlay: {
        value: '700',
        target: 'web',
        comment: 'Default z-index for modal and popup overlays.',
        filePath: './src/tokens/zindex.json',
        isSource: true,
        original: {
          value: '700',
          target: 'web',
          comment: 'Default z-index for modal and popup overlays.'
        },
        name: 'DepthZIndexOverlay',
        attributes: {
          category: 'depth',
          type: 'z-index',
          item: 'overlay'
        },
        path: ['depth', 'z-index', 'overlay']
      },
      loader: {
        value: '800',
        target: 'web',
        comment: 'Default z-index for loaders, stacking on top of overlay.',
        filePath: './src/tokens/zindex.json',
        isSource: true,
        original: {
          value: '800',
          target: 'web',
          comment: 'Default z-index for loaders, stacking on top of overlay.'
        },
        name: 'DepthZIndexLoader',
        attributes: {
          category: 'depth',
          type: 'z-index',
          item: 'loader'
        },
        path: ['depth', 'z-index', 'loader']
      },
      modal: {
        value: '900',
        target: 'web',
        comment:
          'Default z-index for modals that stacks on top of overlays and other elements, but still allows popovers to be visible.',
        filePath: './src/tokens/zindex.json',
        isSource: true,
        original: {
          value: '900',
          target: 'web',
          comment:
            'Default z-index for modals that stacks on top of overlays and other elements, but still allows popovers to be visible.'
        },
        name: 'DepthZIndexModal',
        attributes: {
          category: 'depth',
          type: 'z-index',
          item: 'modal'
        },
        path: ['depth', 'z-index', 'modal']
      },
      popover: {
        value: '950',
        target: 'web',
        comment: 'Default z-index for popover that stacks on top of all other elements.',
        filePath: './src/tokens/zindex.json',
        isSource: true,
        original: {
          value: '950',
          target: 'web',
          comment: 'Default z-index for popover that stacks on top of all other elements.'
        },
        name: 'DepthZIndexPopover',
        attributes: {
          category: 'depth',
          type: 'z-index',
          item: 'popover'
        },
        path: ['depth', 'z-index', 'popover']
      }
    }
  },
  size: {
    2: {
      value: '0.125rem',
      target: 'mobile',
      comment: '2px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '0.125rem',
        target: 'mobile',
        comment: '2px'
      },
      name: 'Size2',
      attributes: {
        category: 'size',
        type: '2'
      },
      path: ['size', '2']
    },
    4: {
      value: '0.25rem',
      target: 'mobile',
      comment: '4px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '0.25rem',
        target: 'mobile',
        comment: '4px'
      },
      name: 'Size4',
      attributes: {
        category: 'size',
        type: '4'
      },
      path: ['size', '4']
    },
    8: {
      value: '0.5rem',
      target: 'mobile',
      comment: '8px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '0.5rem',
        target: 'mobile',
        comment: '8px'
      },
      name: 'Size8',
      attributes: {
        category: 'size',
        type: '8'
      },
      path: ['size', '8']
    },
    12: {
      value: '0.75rem',
      target: 'mobile',
      comment: '12px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '0.75rem',
        target: 'mobile',
        comment: '12px'
      },
      name: 'Size12',
      attributes: {
        category: 'size',
        type: '12'
      },
      path: ['size', '12']
    },
    16: {
      value: '1rem',
      target: 'mobile',
      comment: '16px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '1rem',
        target: 'mobile',
        comment: '16px'
      },
      name: 'Size16',
      attributes: {
        category: 'size',
        type: '16'
      },
      path: ['size', '16']
    },
    20: {
      value: '1.25rem',
      target: 'mobile',
      comment: '20px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '1.25rem',
        target: 'mobile',
        comment: '20px'
      },
      name: 'Size20',
      attributes: {
        category: 'size',
        type: '20'
      },
      path: ['size', '20']
    },
    24: {
      value: '1.5rem',
      target: 'mobile',
      comment: '24px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '1.5rem',
        target: 'mobile',
        comment: '24px'
      },
      name: 'Size24',
      attributes: {
        category: 'size',
        type: '24'
      },
      path: ['size', '24']
    },
    28: {
      value: '1.75rem',
      target: 'mobile',
      comment: '28px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '1.75rem',
        target: 'mobile',
        comment: '28px'
      },
      name: 'Size28',
      attributes: {
        category: 'size',
        type: '28'
      },
      path: ['size', '28']
    },
    32: {
      value: '2rem',
      target: 'mobile',
      comment: '32px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '2rem',
        target: 'mobile',
        comment: '32px'
      },
      name: 'Size32',
      attributes: {
        category: 'size',
        type: '32'
      },
      path: ['size', '32']
    },
    36: {
      value: '2.25rem',
      target: 'mobile',
      comment: '36px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '2.25rem',
        target: 'mobile',
        comment: '36px'
      },
      name: 'Size36',
      attributes: {
        category: 'size',
        type: '36'
      },
      path: ['size', '36']
    },
    40: {
      value: '2.5rem',
      target: 'mobile',
      comment: '40px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '2.5rem',
        target: 'mobile',
        comment: '40px'
      },
      name: 'Size40',
      attributes: {
        category: 'size',
        type: '40'
      },
      path: ['size', '40']
    },
    44: {
      value: '2.75rem',
      target: 'mobile',
      comment: '44px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '2.75rem',
        target: 'mobile',
        comment: '44px'
      },
      name: 'Size44',
      attributes: {
        category: 'size',
        type: '44'
      },
      path: ['size', '44']
    },
    48: {
      value: '3rem',
      target: 'mobile',
      comment: '48px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '3rem',
        target: 'mobile',
        comment: '48px'
      },
      name: 'Size48',
      attributes: {
        category: 'size',
        type: '48'
      },
      path: ['size', '48']
    },
    52: {
      value: '3.25rem',
      target: 'mobile',
      comment: '52px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '3.25rem',
        target: 'mobile',
        comment: '52px'
      },
      name: 'Size52',
      attributes: {
        category: 'size',
        type: '52'
      },
      path: ['size', '52']
    },
    56: {
      value: '3.5rem',
      target: 'mobile',
      comment: '56px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '3.5rem',
        target: 'mobile',
        comment: '56px'
      },
      name: 'Size56',
      attributes: {
        category: 'size',
        type: '56'
      },
      path: ['size', '56']
    },
    60: {
      value: '3.75rem',
      target: 'mobile',
      comment: '60px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '3.75rem',
        target: 'mobile',
        comment: '60px'
      },
      name: 'Size60',
      attributes: {
        category: 'size',
        type: '60'
      },
      path: ['size', '60']
    },
    64: {
      value: '4rem',
      target: 'mobile',
      comment: '64px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '4rem',
        target: 'mobile',
        comment: '64px'
      },
      name: 'Size64',
      attributes: {
        category: 'size',
        type: '64'
      },
      path: ['size', '64']
    },
    68: {
      value: '4.25rem',
      target: 'mobile',
      comment: '68px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '4.25rem',
        target: 'mobile',
        comment: '68px'
      },
      name: 'Size68',
      attributes: {
        category: 'size',
        type: '68'
      },
      path: ['size', '68']
    },
    72: {
      value: '4.5rem',
      target: 'mobile',
      comment: '72px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '4.5rem',
        target: 'mobile',
        comment: '72px'
      },
      name: 'Size72',
      attributes: {
        category: 'size',
        type: '72'
      },
      path: ['size', '72']
    },
    76: {
      value: '4.75rem',
      target: 'mobile',
      comment: '76px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '4.75rem',
        target: 'mobile',
        comment: '76px'
      },
      name: 'Size76',
      attributes: {
        category: 'size',
        type: '76'
      },
      path: ['size', '76']
    },
    80: {
      value: '5rem',
      target: 'mobile',
      comment: '80px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '5rem',
        target: 'mobile',
        comment: '80px'
      },
      name: 'Size80',
      attributes: {
        category: 'size',
        type: '80'
      },
      path: ['size', '80']
    },
    84: {
      value: '5.25rem',
      target: 'mobile',
      comment: '84px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '5.25rem',
        target: 'mobile',
        comment: '84px'
      },
      name: 'Size84',
      attributes: {
        category: 'size',
        type: '84'
      },
      path: ['size', '84']
    },
    88: {
      value: '5.5rem',
      target: 'mobile',
      comment: '88px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '5.5rem',
        target: 'mobile',
        comment: '88px'
      },
      name: 'Size88',
      attributes: {
        category: 'size',
        type: '88'
      },
      path: ['size', '88']
    },
    92: {
      value: '5.75rem',
      target: 'mobile',
      comment: '92px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '5.75rem',
        target: 'mobile',
        comment: '92px'
      },
      name: 'Size92',
      attributes: {
        category: 'size',
        type: '92'
      },
      path: ['size', '92']
    },
    96: {
      value: '6rem',
      target: 'mobile',
      comment: '96px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '6rem',
        target: 'mobile',
        comment: '96px'
      },
      name: 'Size96',
      attributes: {
        category: 'size',
        type: '96'
      },
      path: ['size', '96']
    },
    100: {
      value: '6.25rem',
      target: 'mobile',
      comment: '100px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '6.25rem',
        target: 'mobile',
        comment: '100px'
      },
      name: 'Size100',
      attributes: {
        category: 'size',
        type: '100'
      },
      path: ['size', '100']
    },
    104: {
      value: '6.5rem',
      target: 'mobile',
      comment: '104px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '6.5rem',
        target: 'mobile',
        comment: '104px'
      },
      name: 'Size104',
      attributes: {
        category: 'size',
        type: '104'
      },
      path: ['size', '104']
    },
    120: {
      value: '7.5rem',
      target: 'mobile',
      comment: '120px',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '7.5rem',
        target: 'mobile',
        comment: '120px'
      },
      name: 'Size120',
      attributes: {
        category: 'size',
        type: '120'
      },
      path: ['size', '120']
    },
    '2px': {
      value: '2px',
      target: 'web',
      comment: '0.125rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '2px',
        target: 'web',
        comment: '0.125rem'
      },
      name: 'Size2px',
      attributes: {
        category: 'size',
        type: '2px'
      },
      path: ['size', '2px']
    },
    '4px': {
      value: '4px',
      target: 'web',
      comment: '0.25rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '4px',
        target: 'web',
        comment: '0.25rem'
      },
      name: 'Size4px',
      attributes: {
        category: 'size',
        type: '4px'
      },
      path: ['size', '4px']
    },
    '8px': {
      value: '8px',
      target: 'web',
      comment: '0.5rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '8px',
        target: 'web',
        comment: '0.5rem'
      },
      name: 'Size8px',
      attributes: {
        category: 'size',
        type: '8px'
      },
      path: ['size', '8px']
    },
    '12px': {
      value: '12px',
      target: 'web',
      comment: '0.75rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '12px',
        target: 'web',
        comment: '0.75rem'
      },
      name: 'Size12px',
      attributes: {
        category: 'size',
        type: '12px'
      },
      path: ['size', '12px']
    },
    '16px': {
      value: '16px',
      target: 'web',
      comment: '1rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '16px',
        target: 'web',
        comment: '1rem'
      },
      name: 'Size16px',
      attributes: {
        category: 'size',
        type: '16px'
      },
      path: ['size', '16px']
    },
    '20px': {
      value: '20px',
      target: 'web',
      comment: '1.25rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '20px',
        target: 'web',
        comment: '1.25rem'
      },
      name: 'Size20px',
      attributes: {
        category: 'size',
        type: '20px'
      },
      path: ['size', '20px']
    },
    '24px': {
      value: '24px',
      target: 'web',
      comment: '1.5rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '24px',
        target: 'web',
        comment: '1.5rem'
      },
      name: 'Size24px',
      attributes: {
        category: 'size',
        type: '24px'
      },
      path: ['size', '24px']
    },
    '28px': {
      value: '28px',
      target: 'web',
      comment: '1.75rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '28px',
        target: 'web',
        comment: '1.75rem'
      },
      name: 'Size28px',
      attributes: {
        category: 'size',
        type: '28px'
      },
      path: ['size', '28px']
    },
    '32px': {
      value: '32px',
      target: 'web',
      comment: '2rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '32px',
        target: 'web',
        comment: '2rem'
      },
      name: 'Size32px',
      attributes: {
        category: 'size',
        type: '32px'
      },
      path: ['size', '32px']
    },
    '36px': {
      value: '36px',
      target: 'web',
      comment: '2.25rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '36px',
        target: 'web',
        comment: '2.25rem'
      },
      name: 'Size36px',
      attributes: {
        category: 'size',
        type: '36px'
      },
      path: ['size', '36px']
    },
    '40px': {
      value: '40px',
      target: 'web',
      comment: '2.5rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '40px',
        target: 'web',
        comment: '2.5rem'
      },
      name: 'Size40px',
      attributes: {
        category: 'size',
        type: '40px'
      },
      path: ['size', '40px']
    },
    '44px': {
      value: '44px',
      target: 'web',
      comment: '2.75rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '44px',
        target: 'web',
        comment: '2.75rem'
      },
      name: 'Size44px',
      attributes: {
        category: 'size',
        type: '44px'
      },
      path: ['size', '44px']
    },
    '48px': {
      value: '48px',
      target: 'web',
      comment: '3rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '48px',
        target: 'web',
        comment: '3rem'
      },
      name: 'Size48px',
      attributes: {
        category: 'size',
        type: '48px'
      },
      path: ['size', '48px']
    },
    '52px': {
      value: '52px',
      target: 'web',
      comment: '3.25rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '52px',
        target: 'web',
        comment: '3.25rem'
      },
      name: 'Size52px',
      attributes: {
        category: 'size',
        type: '52px'
      },
      path: ['size', '52px']
    },
    '56px': {
      value: '56px',
      target: 'web',
      comment: '3.5rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '56px',
        target: 'web',
        comment: '3.5rem'
      },
      name: 'Size56px',
      attributes: {
        category: 'size',
        type: '56px'
      },
      path: ['size', '56px']
    },
    '60px': {
      value: '60px',
      target: 'web',
      comment: '3.75rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '60px',
        target: 'web',
        comment: '3.75rem'
      },
      name: 'Size60px',
      attributes: {
        category: 'size',
        type: '60px'
      },
      path: ['size', '60px']
    },
    '64px': {
      value: '64px',
      target: 'web',
      comment: '4rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '64px',
        target: 'web',
        comment: '4rem'
      },
      name: 'Size64px',
      attributes: {
        category: 'size',
        type: '64px'
      },
      path: ['size', '64px']
    },
    '68px': {
      value: '68px',
      target: 'web',
      comment: '4.25rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '68px',
        target: 'web',
        comment: '4.25rem'
      },
      name: 'Size68px',
      attributes: {
        category: 'size',
        type: '68px'
      },
      path: ['size', '68px']
    },
    '72px': {
      value: '72px',
      target: 'web',
      comment: '4.5rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '72px',
        target: 'web',
        comment: '4.5rem'
      },
      name: 'Size72px',
      attributes: {
        category: 'size',
        type: '72px'
      },
      path: ['size', '72px']
    },
    '76px': {
      value: '76px',
      target: 'web',
      comment: '4.75rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '76px',
        target: 'web',
        comment: '4.75rem'
      },
      name: 'Size76px',
      attributes: {
        category: 'size',
        type: '76px'
      },
      path: ['size', '76px']
    },
    '80px': {
      value: '80px',
      target: 'web',
      comment: '5rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '80px',
        target: 'web',
        comment: '5rem'
      },
      name: 'Size80px',
      attributes: {
        category: 'size',
        type: '80px'
      },
      path: ['size', '80px']
    },
    '84px': {
      value: '84px',
      target: 'web',
      comment: '5.25rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '84px',
        target: 'web',
        comment: '5.25rem'
      },
      name: 'Size84px',
      attributes: {
        category: 'size',
        type: '84px'
      },
      path: ['size', '84px']
    },
    '88px': {
      value: '88px',
      target: 'web',
      comment: '5.5rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '88px',
        target: 'web',
        comment: '5.5rem'
      },
      name: 'Size88px',
      attributes: {
        category: 'size',
        type: '88px'
      },
      path: ['size', '88px']
    },
    '92px': {
      value: '92px',
      target: 'web',
      comment: '5.75rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '92px',
        target: 'web',
        comment: '5.75rem'
      },
      name: 'Size92px',
      attributes: {
        category: 'size',
        type: '92px'
      },
      path: ['size', '92px']
    },
    '96px': {
      value: '96px',
      target: 'web',
      comment: '6rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '96px',
        target: 'web',
        comment: '6rem'
      },
      name: 'Size96px',
      attributes: {
        category: 'size',
        type: '96px'
      },
      path: ['size', '96px']
    },
    '100px': {
      value: '100px',
      target: 'web',
      comment: '6.25rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '100px',
        target: 'web',
        comment: '6.25rem'
      },
      name: 'Size100px',
      attributes: {
        category: 'size',
        type: '100px'
      },
      path: ['size', '100px']
    },
    '104px': {
      value: '104px',
      target: 'web',
      comment: '6.5rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '104px',
        target: 'web',
        comment: '6.5rem'
      },
      name: 'Size104px',
      attributes: {
        category: 'size',
        type: '104px'
      },
      path: ['size', '104px']
    },
    '120px': {
      value: '120px',
      target: 'web',
      comment: '7.5rem',
      filePath: './src/tokens/size.json',
      isSource: true,
      original: {
        value: '120px',
        target: 'web',
        comment: '7.5rem'
      },
      name: 'Size120px',
      attributes: {
        category: 'size',
        type: '120px'
      },
      path: ['size', '120px']
    },
    spacing: {
      2: {
        value: '0.125rem',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.2.value}'
        },
        name: 'SizeSpacing2',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '2'
        },
        path: ['size', 'spacing', '2']
      },
      4: {
        value: '0.25rem',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.4.value}'
        },
        name: 'SizeSpacing4',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '4'
        },
        path: ['size', 'spacing', '4']
      },
      8: {
        value: '0.5rem',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.8.value}'
        },
        name: 'SizeSpacing8',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '8'
        },
        path: ['size', 'spacing', '8']
      },
      12: {
        value: '0.75rem',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.12.value}'
        },
        name: 'SizeSpacing12',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '12'
        },
        path: ['size', 'spacing', '12']
      },
      16: {
        value: '1rem',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.16.value}'
        },
        name: 'SizeSpacing16',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '16'
        },
        path: ['size', 'spacing', '16']
      },
      24: {
        value: '1.5rem',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.24.value}'
        },
        name: 'SizeSpacing24',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '24'
        },
        path: ['size', 'spacing', '24']
      },
      32: {
        value: '2rem',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.32.value}'
        },
        name: 'SizeSpacing32',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '32'
        },
        path: ['size', 'spacing', '32']
      },
      40: {
        value: '2.5rem',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.40.value}'
        },
        name: 'SizeSpacing40',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '40'
        },
        path: ['size', 'spacing', '40']
      },
      48: {
        value: '3rem',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.48.value}'
        },
        name: 'SizeSpacing48',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '48'
        },
        path: ['size', 'spacing', '48']
      },
      56: {
        value: '3.5rem',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.56.value}'
        },
        name: 'SizeSpacing56',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '56'
        },
        path: ['size', 'spacing', '56']
      },
      64: {
        value: '4rem',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.64.value}'
        },
        name: 'SizeSpacing64',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '64'
        },
        path: ['size', 'spacing', '64']
      },
      88: {
        value: '5.5rem',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.88.value}'
        },
        name: 'SizeSpacing88',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '88'
        },
        path: ['size', 'spacing', '88']
      },
      120: {
        value: '7.5rem',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.120.value}'
        },
        name: 'SizeSpacing120',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '120'
        },
        path: ['size', 'spacing', '120']
      },
      '2px': {
        value: '2px',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.2px.value}'
        },
        name: 'SizeSpacing2px',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '2px'
        },
        path: ['size', 'spacing', '2px']
      },
      '4px': {
        value: '4px',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.4px.value}'
        },
        name: 'SizeSpacing4px',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '4px'
        },
        path: ['size', 'spacing', '4px']
      },
      '8px': {
        value: '8px',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.8px.value}'
        },
        name: 'SizeSpacing8px',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '8px'
        },
        path: ['size', 'spacing', '8px']
      },
      '12px': {
        value: '12px',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.12px.value}'
        },
        name: 'SizeSpacing12px',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '12px'
        },
        path: ['size', 'spacing', '12px']
      },
      '16px': {
        value: '16px',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.16px.value}'
        },
        name: 'SizeSpacing16px',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '16px'
        },
        path: ['size', 'spacing', '16px']
      },
      '24px': {
        value: '24px',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.24px.value}'
        },
        name: 'SizeSpacing24px',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '24px'
        },
        path: ['size', 'spacing', '24px']
      },
      '32px': {
        value: '32px',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.32px.value}'
        },
        name: 'SizeSpacing32px',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '32px'
        },
        path: ['size', 'spacing', '32px']
      },
      '40px': {
        value: '40px',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.40px.value}'
        },
        name: 'SizeSpacing40px',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '40px'
        },
        path: ['size', 'spacing', '40px']
      },
      '48px': {
        value: '48px',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.48px.value}'
        },
        name: 'SizeSpacing48px',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '48px'
        },
        path: ['size', 'spacing', '48px']
      },
      '56px': {
        value: '56px',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.56px.value}'
        },
        name: 'SizeSpacing56px',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '56px'
        },
        path: ['size', 'spacing', '56px']
      },
      '64px': {
        value: '64px',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.64px.value}'
        },
        name: 'SizeSpacing64px',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '64px'
        },
        path: ['size', 'spacing', '64px']
      },
      '88px': {
        value: '88px',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.88px.value}'
        },
        name: 'SizeSpacing88px',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '88px'
        },
        path: ['size', 'spacing', '88px']
      },
      '120px': {
        value: '120px',
        filePath: './src/tokens/spacing.json',
        isSource: true,
        original: {
          value: '{size.120px.value}'
        },
        name: 'SizeSpacing120px',
        attributes: {
          category: 'size',
          type: 'spacing',
          item: '120px'
        },
        path: ['size', 'spacing', '120px']
      }
    }
  }
};
