// @ids-core/utils/src/variables/js/generated-variables.esm.js
export const IDS_COLOR_BORDER_1 = "rgba(110,98,94,0.2)";
export const IDS_COLOR_BORDER_2 = "rgba(110,98,94,0.3)";
export const IDS_COLOR_BORDER_3 = "rgba(110,98,94,0.5)";
export const IDS_COLOR_BORDER_BASE = "rgb(110, 98, 94)";
export const IDS_SIZE_2 = "0.125rem"; // 2px
export const IDS_SIZE_4 = "0.25rem"; // 4px
export const IDS_SIZE_8 = "0.5rem"; // 8px
export const IDS_SIZE_12 = "0.75rem"; // 12px
export const IDS_SIZE_16 = "1rem"; // 16px
export const IDS_SIZE_20 = "1.25rem"; // 20px
export const IDS_SIZE_24 = "1.5rem"; // 24px
export const IDS_SIZE_28 = "1.75rem"; // 28px
export const IDS_SIZE_32 = "2rem"; // 32px
export const IDS_SIZE_36 = "2.25rem"; // 36px
export const IDS_SIZE_40 = "2.5rem"; // 40px
export const IDS_SIZE_44 = "2.75rem"; // 44px
export const IDS_SIZE_48 = "3rem"; // 48px
export const IDS_SIZE_52 = "3.25rem"; // 52px
export const IDS_SIZE_56 = "3.5rem"; // 56px
export const IDS_SIZE_60 = "3.75rem"; // 60px
export const IDS_SIZE_64 = "4rem"; // 64px
export const IDS_SIZE_68 = "4.25rem"; // 68px
export const IDS_SIZE_72 = "4.5rem"; // 72px
export const IDS_SIZE_76 = "4.75rem"; // 76px
export const IDS_SIZE_80 = "5rem"; // 80px
export const IDS_SIZE_84 = "5.25rem"; // 84px
export const IDS_SIZE_88 = "5.5rem"; // 88px
export const IDS_SIZE_92 = "5.75rem"; // 92px
export const IDS_SIZE_96 = "6rem"; // 96px
export const IDS_SIZE_100 = "6.25rem"; // 100px
export const IDS_SIZE_104 = "6.5rem"; // 104px
export const IDS_SIZE_120 = "7.5rem"; // 120px
export const IDS_SIZE_BORDER_1 = "0.0625rem";
export const IDS_SIZE_BORDER_2 = "0.125rem";
export const IDS_SIZE_BORDER_3 = "0.1875rem";
export const IDS_SIZE_2PX = "2px"; // 0.125rem
export const IDS_SIZE_4PX = "4px"; // 0.25rem
export const IDS_SIZE_8PX = "8px"; // 0.5rem
export const IDS_SIZE_12PX = "12px"; // 0.75rem
export const IDS_SIZE_16PX = "16px"; // 1rem
export const IDS_SIZE_20PX = "20px"; // 1.25rem
export const IDS_SIZE_24PX = "24px"; // 1.5rem
export const IDS_SIZE_28PX = "28px"; // 1.75rem
export const IDS_SIZE_32PX = "32px"; // 2rem
export const IDS_SIZE_36PX = "36px"; // 2.25rem
export const IDS_SIZE_40PX = "40px"; // 2.5rem
export const IDS_SIZE_44PX = "44px"; // 2.75rem
export const IDS_SIZE_48PX = "48px"; // 3rem
export const IDS_SIZE_52PX = "52px"; // 3.25rem
export const IDS_SIZE_56PX = "56px"; // 3.5rem
export const IDS_SIZE_60PX = "60px"; // 3.75rem
export const IDS_SIZE_64PX = "64px"; // 4rem
export const IDS_SIZE_68PX = "68px"; // 4.25rem
export const IDS_SIZE_72PX = "72px"; // 4.5rem
export const IDS_SIZE_76PX = "76px"; // 4.75rem
export const IDS_SIZE_80PX = "80px"; // 5rem
export const IDS_SIZE_84PX = "84px"; // 5.25rem
export const IDS_SIZE_88PX = "88px"; // 5.5rem
export const IDS_SIZE_92PX = "92px"; // 5.75rem
export const IDS_SIZE_96PX = "96px"; // 6rem
export const IDS_SIZE_100PX = "100px"; // 6.25rem
export const IDS_SIZE_104PX = "104px"; // 6.5rem
export const IDS_SIZE_120PX = "120px"; // 7.5rem
export const IDS_SIZE_SPACING_2 = "0.125rem";
export const IDS_SIZE_SPACING_4 = "0.25rem";
export const IDS_SIZE_SPACING_8 = "0.5rem";
export const IDS_SIZE_SPACING_12 = "0.75rem";
export const IDS_SIZE_SPACING_16 = "1rem";
export const IDS_SIZE_SPACING_24 = "1.5rem";
export const IDS_SIZE_SPACING_32 = "2rem";
export const IDS_SIZE_SPACING_40 = "2.5rem";
export const IDS_SIZE_SPACING_48 = "3rem";
export const IDS_SIZE_SPACING_56 = "3.5rem";
export const IDS_SIZE_SPACING_64 = "4rem";
export const IDS_SIZE_SPACING_88 = "5.5rem";
export const IDS_SIZE_SPACING_120 = "7.5rem";
export const IDS_SIZE_SPACING_LARGEST = "5.5rem";
export const IDS_SIZE_SPACING_LARGER = "4rem";
export const IDS_SIZE_SPACING_LARGE = "2.5rem";
export const IDS_SIZE_SPACING_DEFAULT = "2rem";
export const IDS_SIZE_SPACING_SMALL = "1.5rem";
export const IDS_SIZE_SPACING_SMALLER = "1rem";
export const IDS_SIZE_SPACING_SMALLEST = "0.75rem";
export const IDS_SIZE_SPACING_2PX = "2px";
export const IDS_SIZE_SPACING_4PX = "4px";
export const IDS_SIZE_SPACING_8PX = "8px";
export const IDS_SIZE_SPACING_12PX = "12px";
export const IDS_SIZE_SPACING_16PX = "16px";
export const IDS_SIZE_SPACING_24PX = "24px";
export const IDS_SIZE_SPACING_32PX = "32px";
export const IDS_SIZE_SPACING_40PX = "40px";
export const IDS_SIZE_SPACING_48PX = "48px";
export const IDS_SIZE_SPACING_56PX = "56px";
export const IDS_SIZE_SPACING_64PX = "64px";
export const IDS_SIZE_SPACING_88PX = "88px";
export const IDS_SIZE_SPACING_120PX = "120px";
export const IDS_DEPTH_OPACITY_HIDDEN = "0"; // Makes the element completely tranparent.
export const IDS_DEPTH_OPACITY_BORDER = "0.2"; // Gives 20% opacity to an element.
export const IDS_DEPTH_OPACITY_DISABLED = "0.3"; // Gives 30% opacity to an element.
export const IDS_DEPTH_OPACITY_BACKDROP = "0.4"; // Gives 40% opacity to an element.
export const IDS_DEPTH_OPACITY_VISIBLE = "1"; // Makes an element completely visible.
export const IDS_DEPTH_SHADOW_100_LIGHT = "0 8px 4px -2px rgba(110,98,94, 0.08), 0 8px 8px rgba(110,98,94, 0.12)";
export const IDS_DEPTH_SHADOW_100_DARK = "0 8px 4px -2px rgba(51,30,17, 0.08), 0 8px 8px rgba(51,30,17, 0.20)";
export const IDS_DEPTH_SHADOW_200_LIGHT = "0 8px 4px -2px rgba(110,98,94, 0.08), 0 12px 16px rgba(110,98,94, 0.16)";
export const IDS_DEPTH_SHADOW_200_DARK = "0 8px 4px -2px rgba(51,30,17, 0.08), 0 12px 16px rgba(51,30,17, 0.24)";
export const IDS_DEPTH_SHADOW_300_LIGHT = "0 8px 4px -2px rgba(110,98,94, 0.08), 0 16px 24px rgba(110,98,94, 0.20)";
export const IDS_DEPTH_SHADOW_300_DARK = "0 8px 4px -2px rgba(51,30,17, 0.08), 0 16px 24px rgba(51,30,17, 0.28)";
export const IDS_DEPTH_SHADOW_400_LIGHT = "0 8px 4px -2px rgba(110,98,94, 0.08), 0 20px 32px rgba(110,98,94, 0.24)";
export const IDS_DEPTH_SHADOW_400_DARK = "0 8px 4px -2px rgba(51,30,17, 0.08), 0 20px 32px rgba(51,30,17, 0.32)";
export const IDS_DEPTH_Z_INDEX_DEEP = "-999999"; // Deep z-index is used to stack something behind everything else.
export const IDS_DEPTH_Z_INDEX_DEFAULT = "1"; // The default z-index for components and elements inside components.
export const IDS_DEPTH_Z_INDEX_MASKED = "100"; // Default z-index for elements that are beeing masked.
export const IDS_DEPTH_Z_INDEX_MASK = "200"; // Default z-index for elements that are used as a mask.
export const IDS_DEPTH_Z_INDEX_STICKY = "300"; // Default z-index for sticky interface elements.
export const IDS_DEPTH_Z_INDEX_GLOBAL_HEADER = "400"; // Default z-index for the Global Header.
export const IDS_DEPTH_Z_INDEX_TOAST = "500"; // Default z-index for toast messages.
export const IDS_DEPTH_Z_INDEX_DROPDOWN = "600"; // Default z-index for dropdowns which makes sure the dropdown stacks above toasts and sticky elements.
export const IDS_DEPTH_Z_INDEX_OVERLAY = "700"; // Default z-index for modal and popup overlays.
export const IDS_DEPTH_Z_INDEX_LOADER = "800"; // Default z-index for loaders, stacking on top of overlay.
export const IDS_DEPTH_Z_INDEX_MODAL = "900"; // Default z-index for modals that stacks on top of overlays and other elements, but still allows popovers to be visible.
export const IDS_DEPTH_Z_INDEX_POPOVER = "950"; // Default z-index for popover that stacks on top of all other elements.
