// @ids-core/color/src/variables/js/deprecated-variables.module.js
module.exports = {
    color: {
        accent: {
            blue: {
                value: '#0054f0',
                filePath: './src/tokens/accent.json',
                isSource: true,
                original: {
                    value: '{color.base.blue.100.value}',
                },
                name: 'ColorAccentBlue',
                attributes: {
                    category: 'color',
                    type: 'accent',
                    item: 'blue',
                },
                path: ['color', 'accent', 'blue'],
            },
        },
        background: {
            'lightest-beige': {
                value: '#faf9f7',
                filePath: './src/tokens/background.json',
                isSource: true,
                original: {
                    value: '{color.base.beige.100.value}',
                },
                name: 'ColorBackgroundLightestBeige',
                attributes: {
                    category: 'color',
                    type: 'background',
                    item: 'lightest-beige',
                },
                path: ['color', 'background', 'lightest-beige'],
            },
            'lighter-beige': {
                value: '#f6f3f0',
                filePath: './src/tokens/background.json',
                isSource: true,
                original: {
                    value: '{color.base.beige.200.value}',
                },
                name: 'ColorBackgroundLighterBeige',
                attributes: {
                    category: 'color',
                    type: 'background',
                    item: 'lighter-beige',
                },
                path: ['color', 'background', 'lighter-beige'],
            },
            'light-beige': {
                value: '#f1ece8',
                filePath: './src/tokens/background.json',
                isSource: true,
                original: {
                    value: '{color.base.beige.300.value}',
                },
                name: 'ColorBackgroundLightBeige',
                attributes: {
                    category: 'color',
                    type: 'background',
                    item: 'light-beige',
                },
                path: ['color', 'background', 'light-beige'],
            },
            beige: {
                value: '#ede6e1',
                filePath: './src/tokens/background.json',
                isSource: true,
                original: {
                    value: '{color.base.beige.400.value}',
                },
                name: 'ColorBackgroundBeige',
                attributes: {
                    category: 'color',
                    type: 'background',
                    item: 'beige',
                },
                path: ['color', 'background', 'beige'],
            },
            'dark-beige': {
                value: '#e8e0d9',
                filePath: './src/tokens/background.json',
                isSource: true,
                original: {
                    value: '{color.base.beige.500.value}',
                },
                name: 'ColorBackgroundDarkBeige',
                attributes: {
                    category: 'color',
                    type: 'background',
                    item: 'dark-beige',
                },
                path: ['color', 'background', 'dark-beige'],
            },
        },
        base: {
            'light-blue': {
                100: {
                    name: 'ColorBaseLightBlue100',
                    'color-name': 'Gas Giant',
                    value: '#97e2f7',
                    rgb: 'rgb(151, 226, 247)',
                    hex: '#97e2f7',
                    'internal-name': 'CB 3',
                    pmsc: 'PMS 291 C',
                    pmsu: 'PMS 544 U',
                    cmykc: '35, 0, 2, 0',
                    cmyku: '34, 3, 0, 2',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Gas Giant',
                        'color-name': 'Gas Giant',
                        value: 'rgb(151, 226, 247)',
                        rgb: 'rgb(151, 226, 247)',
                        hex: '#97e2f7',
                        'internal-name': 'CB 3',
                        pmsc: 'PMS 291 C',
                        pmsu: 'PMS 544 U',
                        cmykc: '35, 0, 2, 0',
                        cmyku: '34, 3, 0, 2',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'light-blue',
                        subitem: '100',
                    },
                    path: ['color', 'base', 'light-blue', '100'],
                },
                200: {
                    name: 'ColorBaseLightBlue200',
                    'color-name': 'Blue Martini',
                    value: '#4cb0d3',
                    rgb: 'rgb(76, 176, 211)',
                    hex: '#4cb0d3',
                    'internal-name': 'CB 2',
                    pmsc: 'PMS 2389 C',
                    pmsu: 'PMS 2389 U',
                    cmykc: '68, 13, 12, 0',
                    cmyku: '63, 16, 8, 0',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Blue Martini',
                        'color-name': 'Blue Martini',
                        value: 'rgb(76, 176, 211)',
                        rgb: 'rgb(76, 176, 211)',
                        hex: '#4cb0d3',
                        'internal-name': 'CB 2',
                        pmsc: 'PMS 2389 C',
                        pmsu: 'PMS 2389 U',
                        cmykc: '68, 13, 12, 0',
                        cmyku: '63, 16, 8, 0',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'light-blue',
                        subitem: '200',
                    },
                    path: ['color', 'base', 'light-blue', '200'],
                },
                300: {
                    name: 'ColorBaseLightBlue300',
                    'color-name': 'Vanity',
                    value: '#5392b2',
                    rgb: 'rgb(83, 146, 178)',
                    hex: '#5392b2',
                    'internal-name': 'CB 1',
                    pmsc: 'PMS 2159 C',
                    pmsu: 'PMS 2149 U',
                    cmykc: '70, 37, 18, 10',
                    cmyku: '65, 30, 15, 5',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Vanity',
                        'color-name': 'Vanity',
                        value: 'rgb(83, 146, 178)',
                        rgb: 'rgb(83, 146, 178)',
                        hex: '#5392b2',
                        'internal-name': 'CB 1',
                        pmsc: 'PMS 2159 C',
                        pmsu: 'PMS 2149 U',
                        cmykc: '70, 37, 18, 10',
                        cmyku: '65, 30, 15, 5',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'light-blue',
                        subitem: '300',
                    },
                    path: ['color', 'base', 'light-blue', '300'],
                },
            },
            blue: {
                100: {
                    name: 'ColorBaseBlue100',
                    'color-name': 'Megaman Helmet',
                    value: '#0054f0',
                    rgb: 'rgb(0,84,240)',
                    hex: '#0054f0',
                    'internal-name': 'BL 1',
                    pms: 'PMS 3005',
                    cmyk: '100,30,0,0',
                    ncs: 'S 2565-R80B',
                    ral: '270 40 40 Zauberblau',
                    tcx: '18-4051 TCX',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Megaman Helmet',
                        'color-name': 'Megaman Helmet',
                        value: 'rgb(0,84,240)',
                        rgb: 'rgb(0,84,240)',
                        hex: '#0054f0',
                        'internal-name': 'BL 1',
                        pms: 'PMS 3005',
                        cmyk: '100,30,0,0',
                        ncs: 'S 2565-R80B',
                        ral: '270 40 40 Zauberblau',
                        tcx: '18-4051 TCX',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'blue',
                        subitem: '100',
                    },
                    path: ['color', 'base', 'blue', '100'],
                },
                200: {
                    name: 'ColorBaseBlue200',
                    'color-name': 'Bracing Blue',
                    value: '#004080',
                    rgb: 'rgb(0, 64, 128)',
                    hex: '#004080',
                    'internal-name': 'BL 1 Dark',
                    cmyk: '100,50,0,50',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Bracing Blue',
                        'color-name': 'Bracing Blue',
                        value: 'rgb(0, 64, 128)',
                        rgb: 'rgb(0, 64, 128)',
                        hex: '#004080',
                        'internal-name': 'BL 1 Dark',
                        cmyk: '100,50,0,50',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'blue',
                        subitem: '200',
                    },
                    path: ['color', 'base', 'blue', '200'],
                },
            },
            orange: {
                100: {
                    name: 'ColorBaseOrange100',
                    'color-name': 'Wiener Schnitzel',
                    value: '#f09c00',
                    rgb: 'rgb(240, 156, 0)',
                    hex: '#f09c00',
                    'internal-name': 'BL 1 Complement, Deprecated',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Wiener Schnitzel',
                        'color-name': 'Wiener Schnitzel',
                        value: 'rgb(240, 156, 0)',
                        rgb: 'rgb(240, 156, 0)',
                        hex: '#f09c00',
                        'internal-name': 'BL 1 Complement, Deprecated',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'orange',
                        subitem: '100',
                    },
                    path: ['color', 'base', 'orange', '100'],
                },
            },
            beige: {
                100: {
                    name: 'ColorBaseBeige100',
                    'color-name': 'Doctor',
                    value: '#faf9f7',
                    rgb: 'rgb(250,249,247)',
                    hex: '#faf9f7',
                    'internal-name': 'BE 5',
                    pms: 'PMS 9223 / 20%',
                    cmykc: '5, 10, 15, 3 / 20%',
                    cmyku: '3, 8, 14, 2 / 20%',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Doctor',
                        'color-name': 'Doctor',
                        value: 'rgb(250,249,247)',
                        rgb: 'rgb(250,249,247)',
                        hex: '#faf9f7',
                        'internal-name': 'BE 5',
                        pms: 'PMS 9223 / 20%',
                        cmykc: '5, 10, 15, 3 / 20%',
                        cmyku: '3, 8, 14, 2 / 20%',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'beige',
                        subitem: '100',
                    },
                    path: ['color', 'base', 'beige', '100'],
                },
                200: {
                    name: 'ColorBaseBeige200',
                    'color-name': 'The Speed Of Light',
                    value: '#f6f3f0',
                    rgb: 'rgb(246,243,240)',
                    hex: '#f6f3f0',
                    'internal-name': 'BE 4',
                    pms: 'PMS 9223 / 40%',
                    cmykc: '5, 10, 15, 3 / 40%',
                    cmyku: '3, 8, 14, 2 / 40%',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'The Speed Of Light',
                        'color-name': 'The Speed Of Light',
                        value: 'rgb(246,243,240)',
                        rgb: 'rgb(246,243,240)',
                        hex: '#f6f3f0',
                        'internal-name': 'BE 4',
                        pms: 'PMS 9223 / 40%',
                        cmykc: '5, 10, 15, 3 / 40%',
                        cmyku: '3, 8, 14, 2 / 40%',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'beige',
                        subitem: '200',
                    },
                    path: ['color', 'base', 'beige', '200'],
                },
                300: {
                    name: 'ColorBaseBeige300',
                    'color-name': 'Valhallan Blizzard',
                    value: '#f1ece8',
                    rgb: 'rgb(241,236,232)',
                    hex: '#f1ece8',
                    'internal-name': 'BE 3',
                    pms: 'PMS 9223 / 60%',
                    cmykc: '5, 10, 15, 3 / 60%',
                    cmyku: '3, 8, 14, 2 / 60%',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Valhallan Blizzard',
                        'color-name': 'Valhallan Blizzard',
                        value: 'rgb(241,236,232)',
                        rgb: 'rgb(241,236,232)',
                        hex: '#f1ece8',
                        'internal-name': 'BE 3',
                        pms: 'PMS 9223 / 60%',
                        cmykc: '5, 10, 15, 3 / 60%',
                        cmyku: '3, 8, 14, 2 / 60%',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'beige',
                        subitem: '300',
                    },
                    path: ['color', 'base', 'beige', '300'],
                },
                400: {
                    name: 'ColorBaseBeige400',
                    'color-name': 'Desert Storm',
                    value: '#ede6e1',
                    rgb: 'rgb(237,230,225)',
                    hex: '#ede6e1',
                    'internal-name': 'BE 2',
                    pmsc: 'PMS 9223 / 80%',
                    cmykc: '5, 10, 15, 3 / 80%',
                    cmyku: '3, 8, 14, 2 / 80%',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Desert Storm',
                        'color-name': 'Desert Storm',
                        value: 'rgb(237,230,225)',
                        rgb: 'rgb(237,230,225)',
                        hex: '#ede6e1',
                        'internal-name': 'BE 2',
                        pmsc: 'PMS 9223 / 80%',
                        cmykc: '5, 10, 15, 3 / 80%',
                        cmyku: '3, 8, 14, 2 / 80%',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'beige',
                        subitem: '400',
                    },
                    path: ['color', 'base', 'beige', '400'],
                },
                500: {
                    name: 'ColorBaseBeige500',
                    'color-name': 'Stone Harbour',
                    value: '#e8e0d9',
                    rgb: 'rgb(232,224,217)',
                    hex: '#e8e0d9',
                    'internal-name': 'BE 1',
                    pms: 'PMS 9223',
                    cmykc: '5, 10, 15, 3',
                    cmyku: '3, 8, 14, 2',
                    ncs: '1005-Y60R',
                    ral: '060 85 05 Champagnerosé',
                    tcx: '12-0000 TCX',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Stone Harbour',
                        'color-name': 'Stone Harbour',
                        value: 'rgb(232,224,217)',
                        rgb: 'rgb(232,224,217)',
                        hex: '#e8e0d9',
                        'internal-name': 'BE 1',
                        pms: 'PMS 9223',
                        cmykc: '5, 10, 15, 3',
                        cmyku: '3, 8, 14, 2',
                        ncs: '1005-Y60R',
                        ral: '060 85 05 Champagnerosé',
                        tcx: '12-0000 TCX',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'beige',
                        subitem: '500',
                    },
                    path: ['color', 'base', 'beige', '500'],
                },
            },
            pink: {
                100: {
                    name: 'ColorBasePink100',
                    'color-name': 'Peach Crayon',
                    value: '#ffc9a6',
                    rgb: 'rgb(255, 201, 166)',
                    hex: '#ffc9a6',
                    'internal-name': 'CR 3',
                    pmsc: 'PMS 7520 C',
                    pmsu: 'PMS 7520 U',
                    cmykc: '0, 20, 25, 5',
                    cmyku: '5, 17, 31, 0',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Peach Crayon',
                        'color-name': 'Peach Crayon',
                        value: 'rgb(255, 201, 166)',
                        rgb: 'rgb(255, 201, 166)',
                        hex: '#ffc9a6',
                        'internal-name': 'CR 3',
                        pmsc: 'PMS 7520 C',
                        pmsu: 'PMS 7520 U',
                        cmykc: '0, 20, 25, 5',
                        cmyku: '5, 17, 31, 0',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'pink',
                        subitem: '100',
                    },
                    path: ['color', 'base', 'pink', '100'],
                },
                200: {
                    name: 'ColorBasePink200',
                    'color-name': 'Fresh Salmon',
                    value: '#ff8569',
                    rgb: 'rgb(255, 133, 105)',
                    hex: '#ff8569',
                    'internal-name': 'CR 2',
                    pmsc: 'PMS 486 C',
                    pmsu: 'PMS 487 U',
                    cmykc: '0, 53, 48, 0',
                    cmyku: '0, 47, 46, 0',
                    ncs: 'S 1050-Y80R',
                    ral: '030 70 40 Flamingorot',
                    tcx: '16-1526 TCX',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Fresh Salmon',
                        'color-name': 'Fresh Salmon',
                        value: 'rgb(255, 133, 105)',
                        rgb: 'rgb(255, 133, 105)',
                        hex: '#ff8569',
                        'internal-name': 'CR 2',
                        pmsc: 'PMS 486 C',
                        pmsu: 'PMS 487 U',
                        cmykc: '0, 53, 48, 0',
                        cmyku: '0, 47, 46, 0',
                        ncs: 'S 1050-Y80R',
                        ral: '030 70 40 Flamingorot',
                        tcx: '16-1526 TCX',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'pink',
                        subitem: '200',
                    },
                    path: ['color', 'base', 'pink', '200'],
                },
                300: {
                    name: 'ColorBasePink300',
                    'color-name': 'Musk Deer',
                    value: '#805c5c',
                    rgb: 'rgb(128, 92, 92)',
                    hex: '#805c5c',
                    'internal-name': 'CR 1',
                    pmsc: 'PMS 7639 C',
                    pmsu: 'PMS 4995 U',
                    cmykc: '41, 62, 48, 7',
                    cmyku: '0, 42, 15, 48',
                    ncs: 'S 5020-R10B',
                    ral: '360 40 15 Rubingrau',
                    tcx: '18-1612 TCX',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Musk Deer',
                        'color-name': 'Musk Deer',
                        value: 'rgb(128, 92, 92)',
                        rgb: 'rgb(128, 92, 92)',
                        hex: '#805c5c',
                        'internal-name': 'CR 1',
                        pmsc: 'PMS 7639 C',
                        pmsu: 'PMS 4995 U',
                        cmykc: '41, 62, 48, 7',
                        cmyku: '0, 42, 15, 48',
                        ncs: 'S 5020-R10B',
                        ral: '360 40 15 Rubingrau',
                        tcx: '18-1612 TCX',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'pink',
                        subitem: '300',
                    },
                    path: ['color', 'base', 'pink', '300'],
                },
            },
            green: {
                100: {
                    name: 'ColorBaseGreen100',
                    'color-name': 'Camarone',
                    value: '#217331',
                    rgb: 'rgb(33, 115, 49)',
                    hex: '#217331',
                    'internal-name': 'green',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Camarone',
                        'color-name': 'Camarone',
                        value: 'rgb(33, 115, 49)',
                        rgb: 'rgb(33, 115, 49)',
                        hex: '#217331',
                        'internal-name': 'green',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'green',
                        subitem: '100',
                    },
                    path: ['color', 'base', 'green', '100'],
                },
            },
            red: {
                100: {
                    name: 'ColorBaseRed100',
                    'color-name': 'Virtual Boy',
                    value: '#bb320c',
                    rgb: 'rgb(187, 50, 12)',
                    hex: '#bb320c',
                    'internal-name': 'red',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Virtual Boy',
                        'color-name': 'Virtual Boy',
                        value: 'rgb(187, 50, 12)',
                        rgb: 'rgb(187, 50, 12)',
                        hex: '#bb320c',
                        'internal-name': 'red',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'red',
                        subitem: '100',
                    },
                    path: ['color', 'base', 'red', '100'],
                },
            },
            brown: {
                100: {
                    name: 'ColorBaseBrown100',
                    'color-name': 'Fozzie Bear',
                    value: '#6e625e',
                    rgb: 'rgb(110,98,94)',
                    hex: '#6e625e',
                    'internal-name': 'LB 1',
                    pms: 'PMS 7531',
                    cmykc: '15, 27, 36, 51',
                    cmyku: '14, 23, 33, 46',
                    ncs: '6010-Y30R',
                    ral: '060 50 10 Sandsteingrau',
                    tcx: '18-1017 TCX',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Fozzie Bear',
                        'color-name': 'Fozzie Bear',
                        value: 'rgb(110,98,94)',
                        rgb: 'rgb(110,98,94)',
                        hex: '#6e625e',
                        'internal-name': 'LB 1',
                        pms: 'PMS 7531',
                        cmykc: '15, 27, 36, 51',
                        cmyku: '14, 23, 33, 46',
                        ncs: '6010-Y30R',
                        ral: '060 50 10 Sandsteingrau',
                        tcx: '18-1017 TCX',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'brown',
                        subitem: '100',
                    },
                    path: ['color', 'base', 'brown', '100'],
                },
                200: {
                    name: 'ColorBaseBrown200',
                    'color-name': 'Black Slug',
                    value: '#331e11',
                    rgb: 'rgb(51,30,17)',
                    hex: '#331e11',
                    'internal-name': 'BR 1',
                    pms: 'PMS 2322',
                    cmykc: '32, 72, 99, 81',
                    cmyku: '25, 70, 70, 80',
                    ncs: 'S 8010-Y50R',
                    ral: '050 30 10 Obsidianbraun',
                    tcx: '19-0814 TCX',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Black Slug',
                        'color-name': 'Black Slug',
                        value: 'rgb(51,30,17)',
                        rgb: 'rgb(51,30,17)',
                        hex: '#331e11',
                        'internal-name': 'BR 1',
                        pms: 'PMS 2322',
                        cmykc: '32, 72, 99, 81',
                        cmyku: '25, 70, 70, 80',
                        ncs: 'S 8010-Y50R',
                        ral: '050 30 10 Obsidianbraun',
                        tcx: '19-0814 TCX',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'brown',
                        subitem: '200',
                    },
                    path: ['color', 'base', 'brown', '200'],
                },
            },
            teal: {
                100: {
                    name: 'ColorBaseTeal100',
                    'color-name': 'Spearmint Water',
                    value: '#b5ede8',
                    rgb: 'rgb(181, 237, 232)',
                    hex: '#b5ede8',
                    'internal-name': 'CG 3',
                    pmsc: 'PMS 7457 C',
                    pmsu: 'PMS 5523 U',
                    cmykc: '25, 0, 13, 0',
                    cmyku: '20, 0, 12, 0',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Spearmint Water',
                        'color-name': 'Spearmint Water',
                        value: 'rgb(181, 237, 232)',
                        rgb: 'rgb(181, 237, 232)',
                        hex: '#b5ede8',
                        'internal-name': 'CG 3',
                        pmsc: 'PMS 7457 C',
                        pmsu: 'PMS 5523 U',
                        cmykc: '25, 0, 13, 0',
                        cmyku: '20, 0, 12, 0',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'teal',
                        subitem: '100',
                    },
                    path: ['color', 'base', 'teal', '100'],
                },
                200: {
                    name: 'ColorBaseTeal200',
                    'color-name': 'Tranquil Teal',
                    value: '#87c7ba',
                    rgb: 'rgb(135, 199, 186)',
                    hex: '#87c7ba',
                    'internal-name': 'CG 2',
                    pmsc: 'PMS 564 C',
                    pmsu: 'PMS 565 U',
                    cmykc: '53, 0, 32, 0',
                    cmyku: '42, 0, 27, 0',
                    ncs: 'S 1030-B50G',
                    ral: '18 80 20 Opalturkis',
                    tcx: '14-5711 TCX',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Tranquil Teal',
                        'color-name': 'Tranquil Teal',
                        value: 'rgb(135, 199, 186)',
                        rgb: 'rgb(135, 199, 186)',
                        hex: '#87c7ba',
                        'internal-name': 'CG 2',
                        pmsc: 'PMS 564 C',
                        pmsu: 'PMS 565 U',
                        cmykc: '53, 0, 32, 0',
                        cmyku: '42, 0, 27, 0',
                        ncs: 'S 1030-B50G',
                        ral: '18 80 20 Opalturkis',
                        tcx: '14-5711 TCX',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'teal',
                        subitem: '200',
                    },
                    path: ['color', 'base', 'teal', '200'],
                },
                300: {
                    name: 'ColorBaseTeal300',
                    'color-name': 'Little League',
                    value: '#6b9994',
                    rgb: 'rgb(107, 153, 148)',
                    hex: '#6b9994',
                    'internal-name': 'CG 1',
                    pmsc: 'PMS 2211 C',
                    pmsu: 'PMS 5493 U',
                    cmykc: '63, 25, 35, 5',
                    cmyku: '47, 6, 20, 20',
                    ncs: 'S 4020-B30G',
                    ral: '200 60 15 Waidindigo',
                    tcx: '17-5110 TCX',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Little League',
                        'color-name': 'Little League',
                        value: 'rgb(107, 153, 148)',
                        rgb: 'rgb(107, 153, 148)',
                        hex: '#6b9994',
                        'internal-name': 'CG 1',
                        pmsc: 'PMS 2211 C',
                        pmsu: 'PMS 5493 U',
                        cmykc: '63, 25, 35, 5',
                        cmyku: '47, 6, 20, 20',
                        ncs: 'S 4020-B30G',
                        ral: '200 60 15 Waidindigo',
                        tcx: '17-5110 TCX',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'teal',
                        subitem: '300',
                    },
                    path: ['color', 'base', 'teal', '300'],
                },
            },
            yellow: {
                100: {
                    name: 'ColorBaseYellow100',
                    'color-name': 'Swedish Yellow',
                    value: '#ffe082',
                    rgb: 'rgb(255, 224, 130)',
                    hex: '#ffe082',
                    'internal-name': 'CY 3',
                    pmsc: 'PMS 7402 C',
                    pmsu: 'PMS 2001 U',
                    cmykc: '0, 6, 53, 0',
                    cmyku: '0, 3, 48, 0',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Swedish Yellow',
                        'color-name': 'Swedish Yellow',
                        value: 'rgb(255, 224, 130)',
                        rgb: 'rgb(255, 224, 130)',
                        hex: '#ffe082',
                        'internal-name': 'CY 3',
                        pmsc: 'PMS 7402 C',
                        pmsu: 'PMS 2001 U',
                        cmykc: '0, 6, 53, 0',
                        cmyku: '0, 3, 48, 0',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'yellow',
                        subitem: '100',
                    },
                    path: ['color', 'base', 'yellow', '100'],
                },
                200: {
                    name: 'ColorBaseYellow200',
                    'color-name': 'Go Bananas',
                    value: '#fcc74f',
                    rgb: 'rgb(252, 199, 79)',
                    hex: '#fcc74f',
                    'internal-name': 'CY 2',
                    pmsc: 'PMS 141 C',
                    pmsu: 'PMS 7403 U',
                    cmykc: '0, 20, 72, 0',
                    cmyku: '0, 14, 65, 0',
                    ncs: 'S 1050-Y10R',
                    ral: '080 80 60 Vollgelb',
                    tcx: '14-0846 TCX',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Go Bananas',
                        'color-name': 'Go Bananas',
                        value: 'rgb(252, 199, 79)',
                        rgb: 'rgb(252, 199, 79)',
                        hex: '#fcc74f',
                        'internal-name': 'CY 2',
                        pmsc: 'PMS 141 C',
                        pmsu: 'PMS 7403 U',
                        cmykc: '0, 20, 72, 0',
                        cmyku: '0, 14, 65, 0',
                        ncs: 'S 1050-Y10R',
                        ral: '080 80 60 Vollgelb',
                        tcx: '14-0846 TCX',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'yellow',
                        subitem: '200',
                    },
                    path: ['color', 'base', 'yellow', '200'],
                },
                300: {
                    name: 'ColorBaseYellow300',
                    'color-name': 'Apple Cinnamon',
                    value: '#b08759',
                    rgb: 'rgb(176, 135, 89)',
                    hex: '#b08759',
                    'internal-name': 'CY 1',
                    pmsc: 'PMS 2317 C',
                    pmsu: 'PMS 2313 U',
                    cmykc: '35, 48, 70, 5',
                    cmyku: '0, 30, 54, 27',
                    ncs: 'S 4030-Y30R',
                    ral: '060 50 30 Senfbraun',
                    tcx: '17-1125 TCX',
                    filePath: './src/tokens/base.json',
                    isSource: true,
                    original: {
                        name: 'Apple Cinnamon',
                        'color-name': 'Apple Cinnamon',
                        value: 'rgb(176, 135, 89)',
                        rgb: 'rgb(176, 135, 89)',
                        hex: '#b08759',
                        'internal-name': 'CY 1',
                        pmsc: 'PMS 2317 C',
                        pmsu: 'PMS 2313 U',
                        cmykc: '35, 48, 70, 5',
                        cmyku: '0, 30, 54, 27',
                        ncs: 'S 4030-Y30R',
                        ral: '060 50 30 Senfbraun',
                        tcx: '17-1125 TCX',
                    },
                    attributes: {
                        category: 'color',
                        type: 'base',
                        item: 'yellow',
                        subitem: '300',
                    },
                    path: ['color', 'base', 'yellow', '300'],
                },
            },
        },
        cta: {
            value: '#0054f0',
            filePath: './src/tokens/cta.json',
            isSource: true,
            original: {
                value: '{color.accent.blue.value}',
            },
            name: 'ColorCta',
            attributes: {
                category: 'color',
                type: 'cta',
            },
            path: ['color', 'cta'],
        },
        infographic: {
            'dark-red': {
                value: '#805c5c',
                filePath: './src/tokens/infographic.json',
                isSource: true,
                original: {
                    value: '{color.base.pink.300.value}',
                },
                name: 'ColorInfographicDarkRed',
                attributes: {
                    category: 'color',
                    type: 'infographic',
                    item: 'dark-red',
                },
                path: ['color', 'infographic', 'dark-red'],
            },
            red: {
                value: '#ff8569',
                filePath: './src/tokens/infographic.json',
                isSource: true,
                original: {
                    value: '{color.base.pink.200.value}',
                },
                name: 'ColorInfographicRed',
                attributes: {
                    category: 'color',
                    type: 'infographic',
                    item: 'red',
                },
                path: ['color', 'infographic', 'red'],
            },
            'light-red': {
                value: '#ffc9a6',
                filePath: './src/tokens/infographic.json',
                isSource: true,
                original: {
                    value: '{color.base.pink.100.value}',
                },
                name: 'ColorInfographicLightRed',
                attributes: {
                    category: 'color',
                    type: 'infographic',
                    item: 'light-red',
                },
                path: ['color', 'infographic', 'light-red'],
            },
            'dark-yellow': {
                value: '#b08759',
                filePath: './src/tokens/infographic.json',
                isSource: true,
                original: {
                    value: '{color.base.yellow.300.value}',
                },
                name: 'ColorInfographicDarkYellow',
                attributes: {
                    category: 'color',
                    type: 'infographic',
                    item: 'dark-yellow',
                },
                path: ['color', 'infographic', 'dark-yellow'],
            },
            yellow: {
                value: '#fcc74f',
                filePath: './src/tokens/infographic.json',
                isSource: true,
                original: {
                    value: '{color.base.yellow.200.value}',
                },
                name: 'ColorInfographicYellow',
                attributes: {
                    category: 'color',
                    type: 'infographic',
                    item: 'yellow',
                },
                path: ['color', 'infographic', 'yellow'],
            },
            'light-yellow': {
                value: '#ffe082',
                filePath: './src/tokens/infographic.json',
                isSource: true,
                original: {
                    value: '{color.base.yellow.100.value}',
                },
                name: 'ColorInfographicLightYellow',
                attributes: {
                    category: 'color',
                    type: 'infographic',
                    item: 'light-yellow',
                },
                path: ['color', 'infographic', 'light-yellow'],
            },
            'dark-green': {
                value: '#6b9994',
                filePath: './src/tokens/infographic.json',
                isSource: true,
                original: {
                    value: '{color.base.teal.300.value}',
                },
                name: 'ColorInfographicDarkGreen',
                attributes: {
                    category: 'color',
                    type: 'infographic',
                    item: 'dark-green',
                },
                path: ['color', 'infographic', 'dark-green'],
            },
            green: {
                value: '#87c7ba',
                filePath: './src/tokens/infographic.json',
                isSource: true,
                original: {
                    value: '{color.base.teal.200.value}',
                },
                name: 'ColorInfographicGreen',
                attributes: {
                    category: 'color',
                    type: 'infographic',
                    item: 'green',
                },
                path: ['color', 'infographic', 'green'],
            },
            'light-green': {
                value: '#b5ede8',
                filePath: './src/tokens/infographic.json',
                isSource: true,
                original: {
                    value: '{color.base.teal.100.value}',
                },
                name: 'ColorInfographicLightGreen',
                attributes: {
                    category: 'color',
                    type: 'infographic',
                    item: 'light-green',
                },
                path: ['color', 'infographic', 'light-green'],
            },
            'dark-blue': {
                value: '#5392b2',
                filePath: './src/tokens/infographic.json',
                isSource: true,
                original: {
                    value: '{color.base.light-blue.300.value}',
                },
                name: 'ColorInfographicDarkBlue',
                attributes: {
                    category: 'color',
                    type: 'infographic',
                    item: 'dark-blue',
                },
                path: ['color', 'infographic', 'dark-blue'],
            },
            blue: {
                value: '#4cb0d3',
                filePath: './src/tokens/infographic.json',
                isSource: true,
                original: {
                    value: '{color.base.light-blue.200.value}',
                },
                name: 'ColorInfographicBlue',
                attributes: {
                    category: 'color',
                    type: 'infographic',
                    item: 'blue',
                },
                path: ['color', 'infographic', 'blue'],
            },
            'light-blue': {
                value: '#97e2f7',
                filePath: './src/tokens/infographic.json',
                isSource: true,
                original: {
                    value: '{color.base.light-blue.100.value}',
                },
                name: 'ColorInfographicLightBlue',
                attributes: {
                    category: 'color',
                    type: 'infographic',
                    item: 'light-blue',
                },
                path: ['color', 'infographic', 'light-blue'],
            },
        },
        interaction: {
            'dark-blue': {
                value: '#004080',
                filePath: './src/tokens/interaction.json',
                isSource: true,
                original: {
                    value: '{color.base.blue.200.value}',
                },
                name: 'ColorInteractionDarkBlue',
                attributes: {
                    category: 'color',
                    type: 'interaction',
                    item: 'dark-blue',
                },
                path: ['color', 'interaction', 'dark-blue'],
            },
            blue: {
                value: '#0054f0',
                filePath: './src/tokens/interaction.json',
                isSource: true,
                original: {
                    value: '{color.base.blue.100.value}',
                },
                name: 'ColorInteractionBlue',
                attributes: {
                    category: 'color',
                    type: 'interaction',
                    item: 'blue',
                },
                path: ['color', 'interaction', 'blue'],
            },
        },
        shadow: {
            light: {
                value: '#6e625e',
                filePath: './src/tokens/shadow.json',
                isSource: true,
                original: {
                    value: '{color.base.brown.100.value}',
                },
                name: 'ColorShadowLight',
                attributes: {
                    category: 'color',
                    type: 'shadow',
                    item: 'light',
                },
                path: ['color', 'shadow', 'light'],
            },
            dark: {
                value: '#331e11',
                filePath: './src/tokens/shadow.json',
                isSource: true,
                original: {
                    value: '{color.base.brown.200.value}',
                },
                name: 'ColorShadowDark',
                attributes: {
                    category: 'color',
                    type: 'shadow',
                    item: 'dark',
                },
                path: ['color', 'shadow', 'dark'],
            },
        },
        states: {
            hover: {
                value: '#004080',
                filePath: './src/tokens/states.json',
                isSource: true,
                original: {
                    value: '{color.interaction.dark-blue.value}',
                },
                name: 'ColorStatesHover',
                attributes: {
                    category: 'color',
                    type: 'states',
                    item: 'hover',
                },
                path: ['color', 'states', 'hover'],
            },
            focus: {
                value: '#0054f0',
                filePath: './src/tokens/states.json',
                isSource: true,
                original: {
                    value: '{color.interaction.blue.value}',
                },
                name: 'ColorStatesFocus',
                attributes: {
                    category: 'color',
                    type: 'states',
                    item: 'focus',
                },
                path: ['color', 'states', 'focus'],
            },
        },
        status: {
            red: {
                value: '#bb320c',
                filePath: './src/tokens/status.json',
                isSource: true,
                original: {
                    value: '{color.base.red.100.value}',
                },
                name: 'ColorStatusRed',
                attributes: {
                    category: 'color',
                    type: 'status',
                    item: 'red',
                },
                path: ['color', 'status', 'red'],
            },
            green: {
                value: '#217331',
                filePath: './src/tokens/status.json',
                isSource: true,
                original: {
                    value: '{color.base.green.100.value}',
                },
                name: 'ColorStatusGreen',
                attributes: {
                    category: 'color',
                    type: 'status',
                    item: 'green',
                },
                path: ['color', 'status', 'green'],
            },
        },
        text: {
            'lightest-beige': {
                value: '#faf9f7',
                filePath: './src/tokens/text.json',
                isSource: true,
                original: {
                    value: '{color.base.beige.100.value}',
                },
                name: 'ColorTextLightestBeige',
                attributes: {
                    category: 'color',
                    type: 'text',
                    item: 'lightest-beige',
                },
                path: ['color', 'text', 'lightest-beige'],
            },
            'light-brown': {
                value: '#6e625e',
                filePath: './src/tokens/text.json',
                isSource: true,
                original: {
                    value: '{color.base.brown.100.value}',
                },
                name: 'ColorTextLightBrown',
                attributes: {
                    category: 'color',
                    type: 'text',
                    item: 'light-brown',
                },
                path: ['color', 'text', 'light-brown'],
            },
            brown: {
                value: '#331e11',
                filePath: './src/tokens/text.json',
                isSource: true,
                original: {
                    value: '{color.base.brown.200.value}',
                },
                name: 'ColorTextBrown',
                attributes: {
                    category: 'color',
                    type: 'text',
                    item: 'brown',
                },
                path: ['color', 'text', 'brown'],
            },
        },
    },
};
