// @ids-core/color/src/variables/js/deprecated-variables.esm.js
export const ColorAccentBlue = "#0054f0";
export const ColorBackgroundLightestBeige = "#faf9f7";
export const ColorBackgroundLighterBeige = "#f6f3f0";
export const ColorBackgroundLightBeige = "#f1ece8";
export const ColorBackgroundBeige = "#ede6e1";
export const ColorBackgroundDarkBeige = "#e8e0d9";
export const ColorBaseLightBlue100 = "#97e2f7";
export const ColorBaseLightBlue200 = "#4cb0d3";
export const ColorBaseLightBlue300 = "#5392b2";
export const ColorBaseBlue100 = "#0054f0";
export const ColorBaseBlue200 = "#004080";
export const ColorBaseOrange100 = "#f09c00";
export const ColorBaseBeige100 = "#faf9f7";
export const ColorBaseBeige200 = "#f6f3f0";
export const ColorBaseBeige300 = "#f1ece8";
export const ColorBaseBeige400 = "#ede6e1";
export const ColorBaseBeige500 = "#e8e0d9";
export const ColorBasePink100 = "#ffc9a6";
export const ColorBasePink200 = "#ff8569";
export const ColorBasePink300 = "#805c5c";
export const ColorBaseGreen100 = "#217331";
export const ColorBaseRed100 = "#bb320c";
export const ColorBaseBrown100 = "#6e625e";
export const ColorBaseBrown200 = "#331e11";
export const ColorBaseTeal100 = "#b5ede8";
export const ColorBaseTeal200 = "#87c7ba";
export const ColorBaseTeal300 = "#6b9994";
export const ColorBaseYellow100 = "#ffe082";
export const ColorBaseYellow200 = "#fcc74f";
export const ColorBaseYellow300 = "#b08759";
export const ColorCta = "#0054f0";
export const ColorInfographicDarkRed = "#805c5c";
export const ColorInfographicRed = "#ff8569";
export const ColorInfographicLightRed = "#ffc9a6";
export const ColorInfographicDarkYellow = "#b08759";
export const ColorInfographicYellow = "#fcc74f";
export const ColorInfographicLightYellow = "#ffe082";
export const ColorInfographicDarkGreen = "#6b9994";
export const ColorInfographicGreen = "#87c7ba";
export const ColorInfographicLightGreen = "#b5ede8";
export const ColorInfographicDarkBlue = "#5392b2";
export const ColorInfographicBlue = "#4cb0d3";
export const ColorInfographicLightBlue = "#97e2f7";
export const ColorInteractionDarkBlue = "#004080";
export const ColorInteractionBlue = "#0054f0";
export const ColorShadowLight = "#6e625e";
export const ColorShadowDark = "#331e11";
export const ColorStatesHover = "#004080";
export const ColorStatesFocus = "#0054f0";
export const ColorStatusRed = "#bb320c";
export const ColorStatusGreen = "#217331";
export const ColorTextLightestBeige = "#faf9f7";
export const ColorTextLightBrown = "#6e625e";
export const ColorTextBrown = "#331e11";
