// @ids-core/color/src/variables/js/generated-variables.esm.js
export const IDS_COLOR_ACCENT = "rgb(0, 84, 240)";
export const IDS_COLOR_BACKGROUND_100 = "rgb(250, 249, 247)";
export const IDS_COLOR_BACKGROUND_200 = "rgb(246, 243, 240)";
export const IDS_COLOR_BACKGROUND_300 = "rgb(241, 236, 232)";
export const IDS_COLOR_BACKGROUND_400 = "rgb(237, 230, 225)";
export const IDS_COLOR_BACKGROUND_500 = "rgb(232, 224, 217)";
export const IDS_COLOR_LIGHT_BLUE_100 = "rgb(151, 226, 247)";
export const IDS_COLOR_LIGHT_BLUE_200 = "rgb(76, 176, 211)";
export const IDS_COLOR_LIGHT_BLUE_300 = "rgb(83, 146, 178)";
export const IDS_COLOR_BLUE_100 = "rgb(0, 84, 240)";
export const IDS_COLOR_BLUE_200 = "rgb(0, 64, 128)";
export const IDS_COLOR_ORANGE_100 = "rgb(240, 156, 0)";
export const IDS_COLOR_BEIGE_100 = "rgb(250, 249, 247)";
export const IDS_COLOR_BEIGE_200 = "rgb(246, 243, 240)";
export const IDS_COLOR_BEIGE_300 = "rgb(241, 236, 232)";
export const IDS_COLOR_BEIGE_400 = "rgb(237, 230, 225)";
export const IDS_COLOR_BEIGE_500 = "rgb(232, 224, 217)";
export const IDS_COLOR_PINK_100 = "rgb(255, 201, 166)";
export const IDS_COLOR_PINK_200 = "rgb(255, 133, 105)";
export const IDS_COLOR_PINK_300 = "rgb(128, 92, 92)";
export const IDS_COLOR_GREEN_100 = "rgb(33, 115, 49)";
export const IDS_COLOR_RED_100 = "rgb(187, 50, 12)";
export const IDS_COLOR_BROWN_100 = "rgb(110, 98, 94)";
export const IDS_COLOR_BROWN_200 = "rgb(51, 30, 17)";
export const IDS_COLOR_TEAL_100 = "rgb(181, 237, 232)";
export const IDS_COLOR_TEAL_200 = "rgb(135, 199, 186)";
export const IDS_COLOR_TEAL_300 = "rgb(107, 153, 148)";
export const IDS_COLOR_YELLOW_100 = "rgb(255, 224, 130)";
export const IDS_COLOR_YELLOW_200 = "rgb(252, 199, 79)";
export const IDS_COLOR_YELLOW_300 = "rgb(176, 135, 89)";
export const IDS_COLOR_CTA = "rgb(0, 84, 240)";
export const IDS_COLOR_INFOGRAPHIC_100 = "rgb(255, 201, 166)";
export const IDS_COLOR_INFOGRAPHIC_200 = "rgb(255, 133, 105)";
export const IDS_COLOR_INFOGRAPHIC_300 = "rgb(128, 92, 92)";
export const IDS_COLOR_INFOGRAPHIC_400 = "rgb(255, 224, 130)";
export const IDS_COLOR_INFOGRAPHIC_500 = "rgb(252, 199, 79)";
export const IDS_COLOR_INFOGRAPHIC_600 = "rgb(176, 135, 89)";
export const IDS_COLOR_INFOGRAPHIC_700 = "rgb(181, 237, 232)";
export const IDS_COLOR_INFOGRAPHIC_800 = "rgb(135, 199, 186)";
export const IDS_COLOR_INFOGRAPHIC_900 = "rgb(107, 153, 148)";
export const IDS_COLOR_INFOGRAPHIC_1000 = "rgb(151, 226, 247)";
export const IDS_COLOR_INFOGRAPHIC_1100 = "rgb(76, 176, 211)";
export const IDS_COLOR_INFOGRAPHIC_1200 = "rgb(83, 146, 178)";
export const IDS_COLOR_SHADOW_LIGHT = "rgb(110, 98, 94)";
export const IDS_COLOR_SHADOW_DARK = "rgb(51, 30, 17)";
export const IDS_COLOR_STATES_DISABLED = "rgba(110,98,94,0.3)";
export const IDS_COLOR_STATES_ACTIVE = "rgb(0, 64, 128)";
export const IDS_COLOR_STATES_HOVER = "rgb(0, 64, 128)";
export const IDS_COLOR_STATES_FOCUS = "rgb(0, 84, 240)";
export const IDS_COLOR_STATUS_ERROR = "rgb(187, 50, 12)";
export const IDS_COLOR_STATUS_WARNING = "rgb(252, 199, 79)";
export const IDS_COLOR_STATUS_SUCCESS = "rgb(33, 115, 49)";
export const IDS_COLOR_TEXT_SUBTLE = "rgb(110, 98, 94)";
export const IDS_COLOR_TEXT_PRIMARY_ON_LIGHT = "rgb(51, 30, 17)";
export const IDS_COLOR_TEXT_PRIMARY_ON_DARK = "rgb(250, 249, 247)";
