// @ids-core/breakpoint/src/variables/js/deprecated-variables.esm.js
export const SizeBreakpointMinXxs = '22.5rem'; // 360px
export const SizeBreakpointMinXs = '25rem'; // 400px
export const SizeBreakpointMinSmlr = '45rem'; // 720px
export const SizeBreakpointMinSm = '60rem'; // 960px
export const SizeBreakpointMinMd = '75rem'; // 1200px
export const SizeBreakpointMinLg = '90rem'; // 1440px
export const SizeBreakpointMinXl = '120rem'; // 1920px
export const SizeBreakpointMinXxl = '160rem'; // 2560px
export const SizeBreakpointMinHuge = '200rem'; // 3200px
export const SizeBreakpointMinHuger = '240rem'; // 3840px
export const SizeBreakpointMaxXxs = '22.4375rem'; // 360px - 1px
export const SizeBreakpointMaxXs = '24.9375rem'; // 400px - 1px
export const SizeBreakpointMaxSmlr = '44.9375rem'; // 720px - 1px
export const SizeBreakpointMaxSm = '59.9375rem'; // 960px - 1px
export const SizeBreakpointMaxMd = '74.9375rem'; // 1200px - 1px
export const SizeBreakpointMaxLg = '89.9375rem'; // 1440px - 1px
export const SizeBreakpointMaxXl = '119.9375rem'; // 1920px - 1px
export const SizeBreakpointMaxXxl = '159.9375rem'; // 2560px - 1px
export const SizeBreakpointMaxHuge = '199.9375rem'; // 3200px - 1px
export const SizeBreakpointMaxHuger = '239.9375rem'; // 3840px - 1px
