// @ids-core/breakpoint/src/variables/js/generated-variables.esm.js



export const IDS_MEDIA_QUERY_XXS_MIN = '22.5rem'; // 360px
export const IDS_MEDIA_QUERY_XS_MIN = '25rem'; // 400px
export const IDS_MEDIA_QUERY_SMLR_MIN = '45rem'; // 720px
export const IDS_MEDIA_QUERY_SM_MIN = '60rem'; // 960px
export const IDS_MEDIA_QUERY_MD_MIN = '75rem'; // 1200px
export const IDS_MEDIA_QUERY_LG_MIN = '90rem'; // 1440px
export const IDS_MEDIA_QUERY_XL_MIN = '120rem'; // 1920px
export const IDS_MEDIA_QUERY_XXL_MIN = '160rem'; // 2560px
export const IDS_MEDIA_QUERY_HUGE_MIN = '200rem'; // 3200px
export const IDS_MEDIA_QUERY_HUGER_MIN = '240rem'; // 3840px
export const IDS_MEDIA_QUERY_XXS_MAX = '22.4375rem'; // 360px - 1px
export const IDS_MEDIA_QUERY_XS_MAX = '24.9375rem'; // 400px - 1px
export const IDS_MEDIA_QUERY_SMLR_MAX = '44.9375rem'; // 720px - 1px
export const IDS_MEDIA_QUERY_SM_MAX = '59.9375rem'; // 960px - 1px
export const IDS_MEDIA_QUERY_MD_MAX = '74.9375rem'; // 1200px - 1px
export const IDS_MEDIA_QUERY_LG_MAX = '89.9375rem'; // 1440px - 1px
export const IDS_MEDIA_QUERY_XL_MAX = '119.9375rem'; // 1920px - 1px
export const IDS_MEDIA_QUERY_XXL_MAX = '159.9375rem'; // 2560px - 1px
export const IDS_MEDIA_QUERY_HUGE_MAX = '199.9375rem'; // 3200px - 1px
export const IDS_MEDIA_QUERY_HUGER_MAX = '239.9375rem'; // 3840px - 1px
