// @ids-core/breakpoint/src/variables/js/variables.umd.js
const generated_variables = require('./generated-variables.umd.js');
/*
 * Deprecated variables
 * @todo remove in version 16
 */
const deprecated_variables = {
    size: {
        breakpoint: {
            'min-xxs': {
                value: '22.5rem',
                name: 'SizeBreakpointMinXxs',
                comment: '360px',
                filePath: './src/tokens/breakpoint.json',
                isSource: true,
                original: {
                    value: '22.5rem',
                    name: 'XXS',
                    comment: '360px',
                },
                attributes: {
                    category: 'size',
                    type: 'breakpoint',
                    item: 'min-xxs',
                },
                path: ['size', 'breakpoint', 'min-xxs'],
            },
            'min-xs': {
                value: '25rem',
                name: 'SizeBreakpointMinXs',
                comment: '400px',
                filePath: './src/tokens/breakpoint.json',
                isSource: true,
                original: {
                    value: '25rem',
                    name: 'ExtraSmall',
                    comment: '400px',
                },
                attributes: {
                    category: 'size',
                    type: 'breakpoint',
                    item: 'min-xs',
                },
                path: ['size', 'breakpoint', 'min-xs'],
            },
            'min-smlr': {
                value: '45rem',
                name: 'SizeBreakpointMinSmlr',
                comment: '720px',
                filePath: './src/tokens/breakpoint.json',
                isSource: true,
                original: {
                    value: '45rem',
                    name: 'Smaller',
                    comment: '720px',
                },
                attributes: {
                    category: 'size',
                    type: 'breakpoint',
                    item: 'min-smlr',
                },
                path: ['size', 'breakpoint', 'min-smlr'],
            },
            'min-sm': {
                value: '60rem',
                name: 'SizeBreakpointMinSm',
                comment: '960px',
                filePath: './src/tokens/breakpoint.json',
                isSource: true,
                original: {
                    value: '60rem',
                    name: 'Small',
                    comment: '960px',
                },
                attributes: {
                    category: 'size',
                    type: 'breakpoint',
                    item: 'min-sm',
                },
                path: ['size', 'breakpoint', 'min-sm'],
            },
            'min-md': {
                value: '75rem',
                name: 'SizeBreakpointMinMd',
                comment: '1200px',
                filePath: './src/tokens/breakpoint.json',
                isSource: true,
                original: {
                    value: '75rem',
                    name: 'Medium',
                    comment: '1200px',
                },
                attributes: {
                    category: 'size',
                    type: 'breakpoint',
                    item: 'min-md',
                },
                path: ['size', 'breakpoint', 'min-md'],
            },
            'min-lg': {
                value: '90rem',
                name: 'SizeBreakpointMinLg',
                comment: '1440px',
                filePath: './src/tokens/breakpoint.json',
                isSource: true,
                original: {
                    value: '90rem',
                    name: 'Large',
                    comment: '1440px',
                },
                attributes: {
                    category: 'size',
                    type: 'breakpoint',
                    item: 'min-lg',
                },
                path: ['size', 'breakpoint', 'min-lg'],
            },
            'min-xl': {
                value: '120rem',
                name: 'SizeBreakpointMinXl',
                comment: '1920px',
                filePath: './src/tokens/breakpoint.json',
                isSource: true,
                original: {
                    value: '120rem',
                    name: 'ExtraLarge',
                    comment: '1920px',
                },
                attributes: {
                    category: 'size',
                    type: 'breakpoint',
                    item: 'min-xl',
                },
                path: ['size', 'breakpoint', 'min-xl'],
            },
            'min-xxl': {
                value: '160rem',
                name: 'SizeBreakpointMinXxl',
                comment: '2560px',
                filePath: './src/tokens/breakpoint.json',
                isSource: true,
                original: {
                    value: '160rem',
                    name: 'Xxl',
                    comment: '2560px',
                },
                attributes: {
                    category: 'size',
                    type: 'breakpoint',
                    item: 'min-xxl',
                },
                path: ['size', 'breakpoint', 'min-xxl'],
            },
            'min-huge': {
                value: '200rem',
                name: 'SizeBreakpointMinHuge',
                comment: '3200px',
                filePath: './src/tokens/breakpoint.json',
                isSource: true,
                original: {
                    value: '200rem',
                    name: 'Huge',
                    comment: '3200px',
                },
                attributes: {
                    category: 'size',
                    type: 'breakpoint',
                    item: 'min-huge',
                },
                path: ['size', 'breakpoint', 'min-huge'],
            },
            'min-huger': {
                value: '240rem',
                name: 'SizeBreakpointMinHuger',
                comment: '3840px',
                filePath: './src/tokens/breakpoint.json',
                isSource: true,
                original: {
                    value: '240rem',
                    name: 'Huger',
                    comment: '3840px',
                },
                attributes: {
                    category: 'size',
                    type: 'breakpoint',
                    item: 'min-huger',
                },
                path: ['size', 'breakpoint', 'min-huger'],
            },
        },
    },
};

(function (root, factory) {
    if (typeof module === 'object' && module.exports) {
        module.exports = factory();
    } else if (typeof exports === 'object') {
        exports['IDS_BREAKPOINT'] = factory();
    } else if (typeof define === 'function' && define.amd) {
        define([], factory);
    } else {
        root['IDS_BREAKPOINT'] = factory();
    }
})(this, function () {
    return { ...deprecated_variables, ...generated_variables };
});
